<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lennart Winkler1, Maria Moiron2, Edward H. Morrow3 and Tim Janicke1,2   1Applied Zoology, Technical University Dresden   2Centre d‚Äô√âcologie Fonctionnelle et √âvolutive, UMR 5175, CNRS, Universit√© de Montpellier 3Department for Environmental and Life Sciences, Karlstad University" />


<title>Stronger net selection on males across animals</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/yeti.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(title);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "Óâô";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "Óâô";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-sm-12 col-md-4 col-lg-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-sm-12 col-md-8 col-lg-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Stronger net selection on males across animals</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Analyses</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Stronger net selection on males across animals</h1>
<h3 class="subtitle">Supplementary material reporting R code</h3>
<h4 class="author">Lennart Winkler<sup>1</sup>, Maria Moiron<sup>2</sup>, Edward H. Morrow<sup>3</sup> and Tim Janicke<sup>1,2</sup> <br></br> <sup>1</sup>Applied Zoology, Technical University Dresden <br> <sup>2</sup>Centre d‚Äô√âcologie Fonctionnelle et √âvolutive, UMR 5175, CNRS, Universit√© de Montpellier</br> <sup>3</sup>Department for Environmental and Life Sciences, Karlstad University</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-06-23
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Net_Selection_eLife_code/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it‚Äôs best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210613code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210613)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210613code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210613)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLennartWinklerNetSelectioneLifecodetree85c0b6e8d8dc0026d92e73098cb42bdb8b49d209targetblank85c0b6ea"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/tree/85c0b6e8d8dc0026d92e73098cb42bdb8b49d209" target="_blank">85c0b6e</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomLennartWinklerNetSelectioneLifecodetree85c0b6e8d8dc0026d92e73098cb42bdb8b49d209targetblank85c0b6ea" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/tree/85c0b6e8d8dc0026d92e73098cb42bdb8b49d209" target="_blank">85c0b6e</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    data/.Rhistory

Untracked files:
    Untracked:  analysis/docs/.nojekyll
    Untracked:  analysis/docs/about.html
    Untracked:  analysis/docs/license.html
    Untracked:  analysis/docs/site_libs/
    Untracked:  data/META_SexSpecGenVar_Data_v23.csv
    Untracked:  data/META_SexSpecGenVar_Pylogeny_v05_NEWICK.txt

</code></pre>
<p>
Note that any generated files, e.g.¬†HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/index.Rmd</code>) and HTML (<code>docs/index.html</code>) files. If you‚Äôve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/85c0b6e8d8dc0026d92e73098cb42bdb8b49d209/analysis/index.Rmd" target="_blank">85c0b6e</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/85c0b6e8d8dc0026d92e73098cb42bdb8b49d209/docs/index.html" target="_blank">85c0b6e</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/4ed93cf9cbeef47676851510b425c05068a55974/docs/index.html" target="_blank">4ed93cf</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/058af7bee1aadb54aced94c34bd68bb541d4fe9c/analysis/index.Rmd" target="_blank">058af7b</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/058af7bee1aadb54aced94c34bd68bb541d4fe9c/docs/index.html" target="_blank">058af7b</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/223df2163ebb54bb9d7b02410a9fcaa9135214a5/docs/index.html" target="_blank">223df21</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/f63b665b10d723a5dd760a11a411f7deecf7d042/analysis/index.Rmd" target="_blank">f63b665</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/f63b665b10d723a5dd760a11a411f7deecf7d042/docs/index.html" target="_blank">f63b665</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/3139eb4d54ec90248d7cb998041d07a29bf51d07/docs/index.html" target="_blank">3139eb4</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/9290105671d5938440c5e00147b7b221b92787d1/analysis/index.Rmd" target="_blank">9290105</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-14
</td>
<td>
wflow_publish(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/9290105671d5938440c5e00147b7b221b92787d1/docs/index.html" target="_blank">9290105</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-14
</td>
<td>
wflow_publish(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/01525f804af4cb282742c6731551ef94df0c4d2c/docs/index.html" target="_blank">01525f8</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/161eee9ca4f35be797a9d11e56fad7e08cdff9b8/analysis/index.Rmd" target="_blank">161eee9</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
wflow_publish(‚ÄúC:/Users/lenna/Desktop/Documents/PhD/2020_META SexSpecGenVar/eLife/Net_Selection_eLife_code/analysis/index.Rmd‚Äù)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/6f60c27281e2554ac17c31a6860265ffc84cc6bb/docs/index.html" target="_blank">6f60c27</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/555a290d73889280bb16552b5911a6335d0e0bc7/analysis/index.Rmd" target="_blank">555a290</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
wflow_publish(‚ÄúC:/Users/lenna/Documents/Net_Selection_eLife_code/analysis/index.Rmd‚Äù)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/9220292c05c265ac4910fef2f664a3db83327342/docs/index.html" target="_blank">9220292</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/15f14fe3c1a44ca94624f55e682db2816b41d055/analysis/index.Rmd" target="_blank">15f14fe</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
wflow_publish(‚ÄúC:/Users/lenna/Documents/Net_Selection_eLife_code/analysis/index.Rmd‚Äù)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/db2f16f4eed457e18792fe29a85af46525caad72/docs/index.html" target="_blank">db2f16f</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/00738afe7fddfe48ff90a166e230dc7ac3615923/analysis/index.Rmd" target="_blank">00738af</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
wflow_publish(‚ÄúC:/Users/lenna/Documents/Net_Selection_eLife_code/analysis/index.Rmd‚Äù)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/7d5ef2c287225924c6124ecb5bccaa216c8a8c56/docs/index.html" target="_blank">7d5ef2c</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/320011ba4a7e729c473d1ce6e756a3bfcfa72046/analysis/index.Rmd" target="_blank">320011b</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Supplementary material reporting R code for the manuscript ‚ÄòStronger net selection on males across animals‚Äô.</p>
<div id="phenotypic-gambit" class="section level2">
<h2>Phenotypic gambit</h2>
<p>Statistical analyses were carried out in two steps. First, we examined the key assumption of the ‚Äòphenotypic gambit‚Äô by testing whether estimates of phenotypic variance predict the estimated genetic variance. For this we computed the Pearson correlation coefficient r, testing the relationship between CVP and CVG for both sexes and the two fitness components separately. In addition, we tested whether the sex bias in CVP translates into a sex bias in CVG by correlating the coefficient of variation ratio lnCVR (Nakagawa et al.¬†2015), which refers to the ln-transformed ratio of male CV to female CV, with positive values indicating a male bias. The analyses on the phenotypic gambit were motivated from a methodological perspective and we did not expect that inter-specific variation in the difference between CVP and CVG can be explained by a shared phylogenetic history. However, for completeness, we also ran correlations on phylogenetic independent contrasts (PICs; computed using the ape R-package (version 5.4.1) in R (Paradis &amp; Schliep 2019)) to test whether our findings were robust when accounting for potential phylogenetic non-independence. We report Pearson‚Äôs correlation coefficients r for normally distributed data and Spearman‚Äôs rho if assumptions of normality were violated.</p>
<div id="load-and-prepare-data" class="section level3">
<h3>Load and prepare data</h3>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" title="1"><span class="co"># load packages</span></a></code></pre></div>
</div>
</div>
<div id="stronger-net-selection-on-males-pglmms" class="section level2">
<h2>Stronger net selection on males (PGLMMs)</h2>
<p>In this second part of the anlysis, we tested the hypothesis that net selection is stronger on males by testing for a male bias in CVP and CVG. Specifically, we ran Phylogenetic General Linear Mixed-Effects Models (PGLMMs) with CVP or CVG as the response variable, and sex as a fixed effect. To account for the paired data structure, we added an observation identifier as a random effect. Moreover, all models included a study identifier and the phylogeny (transformed into a correlation matrix) as random effects to account for statistical non-independence arising from shared study design or phylogenetic history, respectively. Note that the latter also accounts for the non-independence of estimates obtained from the same species as some studies estimated genetic variances from distinct field populations (Fox et al.¬†2004) or different experimental treatments under laboratory conditions such as food stress (Holman &amp; Jacomb 2017) and temperature stress (Berger et al.¬†2014). In an additional series of PGLMMs we tested whether our proxy of sexual selection explained inter-specific variation in the sex-differences of CVP or CVG by adding mating system and its interaction with sex as fixed effects to the models. Finally, given that primary studies varied in the empirical approach used to quantify CVP and CVG, we also used PGLMMs to test whether study type (23 field studies versus 32 laboratory studies) represented a methodological determinant of the observed sex-differences in CVP and CVG. All PGLMMs were ran with the MCMCglmm R-package (version 2.2.9) (Hadfield 2010), using uninformative priors (V = 1, nu = 0.002) and an effective sample size of 20000 (number of iterations = 11000000, burn-in = 1000000, thinning interval = 500). We computed the proportion of variance explained by fixed factors (‚Äòmarginal R2‚Äô) (Nakagawa &amp; Schielzeth 2013). In addition, we quantified the phylogenetic signal as the phylogenetic heritability H2 (i.e., proportional variance in CVP or CVG explained by species identity), which is equivalent to Pagel‚Äôs ùúÜ (de Villemereuil &amp; Nakagawa 2014). In a previous study testing for sex-specific phenotypic variances in reproductive success (Janicke et al.¬†2016), we ran formal meta-analyses using lnCVR as the tested effect size (Nakagawa et al.¬†2015). This is potentially a more powerful approach for comparing phenotypic variances but rendered unsuitable when comparing genetic variances. This is because the computation of the sampling variance of lnCVR is a function of the sample size of the sampled population and the point estimate of lnCVR (Nakagawa et al.¬†2015). However, genetic variances are estimates from statistical models and notorious for being estimated with low precision (i.e.¬†have large confidence intervals). Therefore, using a meta-analytic approach for genetic variances using lnCVR as an effect size leads to overconfident estimation of the global effect size and is therefore likely to result in type-II-errors. However, to allow comparison with the previous meta-analysis, we report the outcome of phylogenetic meta-analyses on phenotypic variances using lnCVR in the Supplementary Material (Table S2), which largely reflects the results on the point estimates of CVP from PGLMMs.</p>
<div id="load-and-prepare-data-1" class="section level3">
<h3>Load and prepare data</h3>
<p>We first load all neccessary packages.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" title="1"><span class="co"># load packages</span></a>
<a class="sourceLine" id="cb2-2" title="2"><span class="kw">rm</span>(<span class="dt">list =</span> <span class="kw">ls</span>())</a>
<a class="sourceLine" id="cb2-3" title="3"><span class="kw">library</span>(ape);<span class="kw">library</span>(metafor); <span class="kw">library</span>(Matrix); <span class="kw">library</span>(MASS); <span class="kw">library</span>(pwr);<span class="kw">library</span>(multcomp);<span class="kw">library</span>(psych);<span class="kw">library</span>(outliers)</a>
<a class="sourceLine" id="cb2-4" title="4"><span class="kw">library</span>(matrixcalc)</a>
<a class="sourceLine" id="cb2-5" title="5"><span class="kw">library</span>(PerformanceAnalytics)</a>
<a class="sourceLine" id="cb2-6" title="6"><span class="kw">library</span>(tidyr)</a>
<a class="sourceLine" id="cb2-7" title="7"><span class="kw">library</span>(MCMCglmm)</a>
<a class="sourceLine" id="cb2-8" title="8"><span class="kw">library</span>(matrixcalc)</a>
<a class="sourceLine" id="cb2-9" title="9"><span class="kw">library</span>(dplyr) </a>
<a class="sourceLine" id="cb2-10" title="10"><span class="kw">library</span>(stargazer)</a>
<a class="sourceLine" id="cb2-11" title="11"><span class="kw">library</span>(data.table)</a>
<a class="sourceLine" id="cb2-12" title="12"><span class="kw">library</span>(ggplot2)</a>
<a class="sourceLine" id="cb2-13" title="13"><span class="kw">library</span>(readr)</a></code></pre></div>
<p>We then load the data set (‚ÄòData‚Äô) and the phylogenetic tree (‚ÄòtheTree‚Äô).</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" title="1"><span class="co"># load data</span></a>
<a class="sourceLine" id="cb3-2" title="2">Data &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">&quot;./data/META_SexSpecGenVar_Data_v23.csv&quot;</span>, <span class="dt">sep=</span><span class="st">&quot;,&quot;</span>, <span class="dt">header=</span><span class="ot">TRUE</span>,<span class="dt">fileEncoding=</span><span class="st">&quot;UTF-8-BOM&quot;</span>)</a>
<a class="sourceLine" id="cb3-3" title="3">theTree &lt;-<span class="st"> </span><span class="kw">read.tree</span>(<span class="st">&quot;./data/META_SexSpecGenVar_Pylogeny_v05_NEWICK.txt&quot;</span>)</a></code></pre></div>
<p>Finally, we subset our data set for the analyses into the fitness categories (‚ÄòFitnessCat‚Äô) lifespan (‚ÄòLS‚Äô) and reproductive success (‚ÄòRS‚Äô).</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" title="1"><span class="co">## Reorganising and subsetting dataset ####</span></a>
<a class="sourceLine" id="cb4-2" title="2"></a>
<a class="sourceLine" id="cb4-3" title="3">stacked_gen_Data &lt;-<span class="st"> </span><span class="kw">gather</span>(Data, <span class="dt">key =</span> <span class="st">&quot;Sex&quot;</span>,<span class="dt">value =</span> <span class="st">&quot;genCV&quot;</span>, genCV_male, genCV_female)</a>
<a class="sourceLine" id="cb4-4" title="4">stacked_phen_Data &lt;-<span class="st"> </span><span class="kw">gather</span>(Data, <span class="dt">key =</span> <span class="st">&quot;Sex&quot;</span>,<span class="dt">value =</span> <span class="st">&quot;phenCV&quot;</span>, phenCV_male, phenCV_female)</a>
<a class="sourceLine" id="cb4-5" title="5"></a>
<a class="sourceLine" id="cb4-6" title="6">RS_gen_metaData&lt;-<span class="kw">subset</span>(stacked_gen_Data, FitnessCat <span class="op">==</span><span class="st"> &quot;RS&quot;</span>)</a>
<a class="sourceLine" id="cb4-7" title="7">LS_gen_metaData&lt;-<span class="kw">subset</span>(stacked_gen_Data, FitnessCat <span class="op">==</span><span class="st"> &quot;LS&quot;</span>)</a>
<a class="sourceLine" id="cb4-8" title="8"></a>
<a class="sourceLine" id="cb4-9" title="9">RS_phen_metaData&lt;-<span class="kw">subset</span>(stacked_phen_Data, FitnessCat <span class="op">==</span><span class="st"> &quot;RS&quot;</span>)</a>
<a class="sourceLine" id="cb4-10" title="10">LS_phen_metaData&lt;-<span class="kw">subset</span>(stacked_phen_Data, FitnessCat <span class="op">==</span><span class="st"> &quot;LS&quot;</span>)</a></code></pre></div>
</div>
<div id="run-mcmc-models" class="section level3">
<h3>Run MCMC models</h3>
<p>We then ran MCMC models to test for differences in the phenotypic and genetic variances in males and females for the fitness categories reproductive success and lifespan.</p>
<p>First we prune the phylogenetic tree to the data subset.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" title="1"><span class="co">## PRUNE PHYLOGENETIC TREE TO DATA SUBSET</span></a>
<a class="sourceLine" id="cb5-2" title="2">RS_gen_metaData<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(RS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb5-3" title="3"><span class="kw">is.factor</span>(RS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb5-4" title="4">RS_Species_Data &lt;-<span class="st"> </span><span class="kw">unique</span>(RS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb5-5" title="5"><span class="kw">summary</span>(RS_Species_Data)</a>
<a class="sourceLine" id="cb5-6" title="6">RS_theTree&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(RS_Species_Data, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb5-7" title="7"><span class="kw">plot</span>(RS_theTree)</a></code></pre></div>
<p><img src="figure/index.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>We then check if the phylogenetic tree was correctly build.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" title="1"><span class="co">## CHECK PHYLOGENETIC TREE</span></a>
<a class="sourceLine" id="cb6-2" title="2"><span class="kw">sort</span>(RS_theTree<span class="op">$</span>tip.label) <span class="op">==</span><span class="st"> </span><span class="kw">sort</span>(<span class="kw">unique</span>(RS_gen_metaData<span class="op">$</span>animal)) <span class="co"># check if tip names correspond to data names</span></a>
<a class="sourceLine" id="cb6-3" title="3"><span class="kw">is.ultrametric</span>(RS_theTree) <span class="co"># check if BL are aligned contemporaneously</span></a>
<a class="sourceLine" id="cb6-4" title="4"><span class="kw">isSymmetric</span>(<span class="kw">vcv</span>(RS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)) <span class="co"># check symmetry of phylogenetic correlation matrix</span></a>
<a class="sourceLine" id="cb6-5" title="5">rawC &lt;-<span class="st"> </span><span class="kw">vcv</span>(RS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb6-6" title="6"><span class="kw">is.positive.definite</span>(rawC) <span class="co"># if FALSE will have to force symmetry</span></a>
<a class="sourceLine" id="cb6-7" title="7">forcedC &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(<span class="kw">forceSymmetric</span>(<span class="kw">vcv</span>(RS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)))</a>
<a class="sourceLine" id="cb6-8" title="8"><span class="kw">is.positive.definite</span>(forcedC)</a>
<a class="sourceLine" id="cb6-9" title="9">comparedC &lt;-<span class="st"> </span>rawC <span class="op">==</span><span class="st"> </span>forcedC</a>
<a class="sourceLine" id="cb6-10" title="10">rawC[<span class="kw">cbind</span>(<span class="kw">which</span>(comparedC<span class="op">!=</span><span class="ot">TRUE</span>, <span class="dt">arr.ind =</span> T))] <span class="op">-</span><span class="st"> </span>forcedC[<span class="kw">cbind</span>(<span class="kw">which</span>(comparedC<span class="op">!=</span><span class="ot">TRUE</span>, <span class="dt">arr.ind =</span> T))] <span class="op">&lt;</span><span class="st"> </span><span class="fl">1e-5</span></a></code></pre></div>
<p>Next, we set the priors for the MCMC models using uninformative priors (V = 1, nu = 0.002) and an effective sample size of 20000 (number of iterations = 11000000, burn-in = 1000000, thinning interval = 500). We also set the index and study ID as factors.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" title="1"><span class="co">## Prior settings, iterations</span></a>
<a class="sourceLine" id="cb7-2" title="2">pr&lt;-<span class="kw">list</span>(<span class="dt">R=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.002</span>), <span class="dt">G=</span><span class="kw">list</span>(<span class="dt">G1=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.002</span>),</a>
<a class="sourceLine" id="cb7-3" title="3">                                      <span class="dt">G2=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.002</span>),</a>
<a class="sourceLine" id="cb7-4" title="4">                                      <span class="dt">G3=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.002</span>)))</a>
<a class="sourceLine" id="cb7-5" title="5"></a>
<a class="sourceLine" id="cb7-6" title="6">BURNIN =<span class="st"> </span><span class="dv">100000</span></a>
<a class="sourceLine" id="cb7-7" title="7">NITT   =<span class="st"> </span><span class="dv">1100000</span></a>
<a class="sourceLine" id="cb7-8" title="8">THIN   =<span class="st"> </span><span class="dv">500</span></a>
<a class="sourceLine" id="cb7-9" title="9"></a>
<a class="sourceLine" id="cb7-10" title="10">BURNIN =<span class="st"> </span><span class="dv">10000</span></a>
<a class="sourceLine" id="cb7-11" title="11">NITT   =<span class="st"> </span><span class="dv">110000</span></a>
<a class="sourceLine" id="cb7-12" title="12">THIN   =<span class="st"> </span><span class="dv">500</span></a>
<a class="sourceLine" id="cb7-13" title="13"></a>
<a class="sourceLine" id="cb7-14" title="14">RS_phen_metaData<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_phen_metaData<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb7-15" title="15">RS_phen_metaData<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_phen_metaData<span class="op">$</span>Study_ID)</a>
<a class="sourceLine" id="cb7-16" title="16">RS_gen_metaData<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_gen_metaData<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb7-17" title="17">RS_gen_metaData<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_gen_metaData<span class="op">$</span>Study_ID)</a></code></pre></div>
<div id="mcmc-models-for-reproductive-success" class="section level4">
<h4>MCMC models for reproductive success</h4>
<p>First, we ran the MCMC testing for overall differences in phenotypic variance in reproductive success (‚ÄòphenCV‚Äô) between males and females (‚ÄòSex‚Äô). The model includes the species (‚Äòanimal‚Äô), estimate ID (‚ÄòIndex‚Äô) and study ID (‚ÄòStudy_ID‚Äô) as random factors.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" title="1">RS_phen_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb8-2" title="2">                          <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb8-3" title="3">                          <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb8-4" title="4">                          <span class="dt">data=</span>RS_phen_metaData,</a>
<a class="sourceLine" id="cb8-5" title="5">                          <span class="dt">pr =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb8-6" title="6">                          <span class="dt">burnin =</span> BURNIN,</a>
<a class="sourceLine" id="cb8-7" title="7">                          <span class="dt">nitt=</span>NITT,</a>
<a class="sourceLine" id="cb8-8" title="8">                          <span class="dt">thin=</span>THIN)</a></code></pre></div>
<pre><code>
                       MCMC iteration = 0

                       MCMC iteration = 1000

                       MCMC iteration = 2000

                       MCMC iteration = 3000

                       MCMC iteration = 4000

                       MCMC iteration = 5000

                       MCMC iteration = 6000

                       MCMC iteration = 7000

                       MCMC iteration = 8000

                       MCMC iteration = 9000

                       MCMC iteration = 10000

                       MCMC iteration = 11000

                       MCMC iteration = 12000

                       MCMC iteration = 13000

                       MCMC iteration = 14000

                       MCMC iteration = 15000

                       MCMC iteration = 16000

                       MCMC iteration = 17000

                       MCMC iteration = 18000

                       MCMC iteration = 19000

                       MCMC iteration = 20000

                       MCMC iteration = 21000

                       MCMC iteration = 22000

                       MCMC iteration = 23000

                       MCMC iteration = 24000

                       MCMC iteration = 25000

                       MCMC iteration = 26000

                       MCMC iteration = 27000

                       MCMC iteration = 28000

                       MCMC iteration = 29000

                       MCMC iteration = 30000

                       MCMC iteration = 31000

                       MCMC iteration = 32000

                       MCMC iteration = 33000

                       MCMC iteration = 34000

                       MCMC iteration = 35000

                       MCMC iteration = 36000

                       MCMC iteration = 37000

                       MCMC iteration = 38000

                       MCMC iteration = 39000

                       MCMC iteration = 40000

                       MCMC iteration = 41000

                       MCMC iteration = 42000

                       MCMC iteration = 43000

                       MCMC iteration = 44000

                       MCMC iteration = 45000

                       MCMC iteration = 46000

                       MCMC iteration = 47000

                       MCMC iteration = 48000

                       MCMC iteration = 49000

                       MCMC iteration = 50000

                       MCMC iteration = 51000

                       MCMC iteration = 52000

                       MCMC iteration = 53000

                       MCMC iteration = 54000

                       MCMC iteration = 55000

                       MCMC iteration = 56000

                       MCMC iteration = 57000

                       MCMC iteration = 58000

                       MCMC iteration = 59000

                       MCMC iteration = 60000

                       MCMC iteration = 61000

                       MCMC iteration = 62000

                       MCMC iteration = 63000

                       MCMC iteration = 64000

                       MCMC iteration = 65000

                       MCMC iteration = 66000

                       MCMC iteration = 67000

                       MCMC iteration = 68000

                       MCMC iteration = 69000

                       MCMC iteration = 70000

                       MCMC iteration = 71000

                       MCMC iteration = 72000

                       MCMC iteration = 73000

                       MCMC iteration = 74000

                       MCMC iteration = 75000

                       MCMC iteration = 76000

                       MCMC iteration = 77000

                       MCMC iteration = 78000

                       MCMC iteration = 79000

                       MCMC iteration = 80000

                       MCMC iteration = 81000

                       MCMC iteration = 82000

                       MCMC iteration = 83000

                       MCMC iteration = 84000

                       MCMC iteration = 85000

                       MCMC iteration = 86000

                       MCMC iteration = 87000

                       MCMC iteration = 88000

                       MCMC iteration = 89000

                       MCMC iteration = 90000

                       MCMC iteration = 91000

                       MCMC iteration = 92000

                       MCMC iteration = 93000

                       MCMC iteration = 94000

                       MCMC iteration = 95000

                       MCMC iteration = 96000

                       MCMC iteration = 97000

                       MCMC iteration = 98000

                       MCMC iteration = 99000

                       MCMC iteration = 100000

                       MCMC iteration = 101000

                       MCMC iteration = 102000

                       MCMC iteration = 103000

                       MCMC iteration = 104000

                       MCMC iteration = 105000

                       MCMC iteration = 106000

                       MCMC iteration = 107000

                       MCMC iteration = 108000

                       MCMC iteration = 109000

                       MCMC iteration = 110000</code></pre>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" title="1"><span class="kw">summary</span>(RS_phen_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 10001:109501
 Thinning interval  = 500
 Sample size  = 200 

 DIC: 50.08103 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.0215 0.0004836  0.08548      200

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.05882 0.001148   0.1242    267.8

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.06306 0.000233   0.1526    200.3

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.06034  0.03996  0.08267    117.8

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC   
(Intercept)               0.6897   0.5147   0.9442      200 &lt;0.005 **
factor(Sex)phenCV_male    0.2316   0.1426   0.2996      200 &lt;0.005 **
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>We then expanded the model to include the matingsytem as a covariate (‚ÄòMating_system‚Äô).</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" title="1">RS_phen_MatSyst_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb12-2" title="2">                                  <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb12-3" title="3">                                  <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb12-4" title="4">                                  <span class="dt">data=</span>RS_phen_metaData,</a>
<a class="sourceLine" id="cb12-5" title="5">                                  <span class="dt">burnin =</span> BURNIN,</a>
<a class="sourceLine" id="cb12-6" title="6">                                  <span class="dt">nitt=</span>NITT,</a>
<a class="sourceLine" id="cb12-7" title="7">                                  <span class="dt">thin=</span>THIN)</a></code></pre></div>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb13-1" title="1"><span class="kw">summary</span>(RS_phen_MatSyst_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 10001:109501
 Thinning interval  = 500
 Sample size  = 200 

 DIC: 31.00689 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.0415 0.0009463   0.1245      200

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.06553 0.0006622   0.1327    345.3

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.05893 0.0002484   0.1553      139

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.05102   0.0327   0.0685      200

 Location effects: phenCV ~ factor(Sex) * factor(Mating_system) 

                                                     post.mean  l-95% CI
(Intercept)                                           0.705011  0.331216
factor(Sex)phenCV_male                               -0.008772 -0.203472
factor(Mating_system)polygamy                         0.008150 -0.297388
factor(Sex)phenCV_male:factor(Mating_system)polygamy  0.319035  0.120912
                                                      u-95% CI eff.samp  pMCMC
(Intercept)                                           1.091192      200   0.01
factor(Sex)phenCV_male                                0.136592      200   0.94
factor(Mating_system)polygamy                         0.489325      200   0.98
factor(Sex)phenCV_male:factor(Mating_system)polygamy  0.483260      200 &lt;0.005
                                                       
(Intercept)                                          * 
factor(Sex)phenCV_male                                 
factor(Mating_system)polygamy                          
factor(Sex)phenCV_male:factor(Mating_system)polygamy **
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>In an additional model, we included the study type (‚ÄòStudyType‚Äô), i.e.¬†if the data were obtained in laboratory or field experiments.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb15-1" title="1">RS_phen_StudyType_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(StudyType),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb15-2" title="2">                                        <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb15-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb15-4" title="4">                                        <span class="dt">data=</span>RS_phen_metaData,</a>
<a class="sourceLine" id="cb15-5" title="5">                                        <span class="dt">burnin =</span> BURNIN,</a>
<a class="sourceLine" id="cb15-6" title="6">                                        <span class="dt">nitt=</span>NITT,</a>
<a class="sourceLine" id="cb15-7" title="7">                                        <span class="dt">thin=</span>THIN)</a></code></pre></div>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb16-1" title="1"><span class="kw">summary</span>(RS_phen_StudyType_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 10001:109501
 Thinning interval  = 500
 Sample size  = 200 

 DIC: 51.0445 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01296 0.0002984  0.04841      200

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.06457 0.005559   0.1166    228.2

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.04607 0.0004687   0.1219      200

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.05913  0.03754  0.08295      200

 Location effects: phenCV ~ factor(Sex) * factor(StudyType) 

                                                   post.mean l-95% CI u-95% CI
(Intercept)                                          0.88229  0.69786  1.13647
factor(Sex)phenCV_male                               0.19596  0.07779  0.32654
factor(StudyType)Laboratory                         -0.32085 -0.60583 -0.04419
factor(Sex)phenCV_male:factor(StudyType)Laboratory   0.06189 -0.11074  0.21244
                                                   eff.samp  pMCMC   
(Intercept)                                           215.5 &lt;0.005 **
factor(Sex)phenCV_male                                200.0   0.01 * 
factor(StudyType)Laboratory                           200.0   0.03 * 
factor(Sex)phenCV_male:factor(StudyType)Laboratory    200.0   0.45   
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC models examining the genetic variance in reproductive success (‚ÄògenCV‚Äô). First, the overall model for a sex difference.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb18-1" title="1">RS_gen_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb18-2" title="2">                       <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb18-3" title="3">                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb18-4" title="4">                       <span class="dt">data=</span>RS_gen_metaData,</a>
<a class="sourceLine" id="cb18-5" title="5">                       <span class="dt">pr =</span> <span class="ot">TRUE</span>,</a>
<a class="sourceLine" id="cb18-6" title="6">                       <span class="dt">burnin =</span> BURNIN,</a>
<a class="sourceLine" id="cb18-7" title="7">                       <span class="dt">nitt=</span>NITT,</a>
<a class="sourceLine" id="cb18-8" title="8">                       <span class="dt">thin=</span>THIN)</a></code></pre></div>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb19-1" title="1"><span class="kw">summary</span>(RS_gen_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 10001:109501
 Thinning interval  = 500
 Sample size  = 200 

 DIC: -130.0802 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01198 0.0004535  0.04331      200

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.003847 0.0003005 0.009868      200

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01471 0.004494  0.02548    248.7

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01497 0.009596  0.02006      200

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp  pMCMC   
(Intercept)             0.24615  0.13462  0.37596    200.0 &lt;0.005 **
factor(Sex)genCV_male   0.08424  0.04133  0.12205    149.8 &lt;0.005 **
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Secondly, including the matingsytem as a covariate (‚ÄòMating_system‚Äô).</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb21-1" title="1">RS_gen_MatSyst_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb21-2" title="2">                  <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb21-3" title="3">                  <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb21-4" title="4">                  <span class="dt">data=</span>RS_gen_metaData,</a>
<a class="sourceLine" id="cb21-5" title="5">                  <span class="dt">burnin =</span> BURNIN,</a>
<a class="sourceLine" id="cb21-6" title="6">                  <span class="dt">nitt=</span>NITT,</a>
<a class="sourceLine" id="cb21-7" title="7">                  <span class="dt">thin=</span>THIN)</a></code></pre></div>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb22-1" title="1"><span class="kw">summary</span>(RS_gen_MatSyst_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 10001:109501
 Thinning interval  = 500
 Sample size  = 200 

 DIC: -138.7594 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01056 0.0002717  0.03152      200

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.004712 0.0004249  0.01062      200

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01545 0.005762  0.03144    156.5

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01356 0.009133  0.01777      200

 Location effects: genCV ~ factor(Sex) * factor(Mating_system) 

                                                    post.mean  l-95% CI
(Intercept)                                          0.254980  0.103623
factor(Sex)genCV_male                               -0.015821 -0.088028
factor(Mating_system)polygamy                       -0.007049 -0.146419
factor(Sex)genCV_male:factor(Mating_system)polygamy  0.135581  0.032283
                                                     u-95% CI eff.samp  pMCMC
(Intercept)                                          0.455024    200.0 &lt;0.005
factor(Sex)genCV_male                                0.071933    200.0   0.74
factor(Mating_system)polygamy                        0.143565    272.4   0.98
factor(Sex)genCV_male:factor(Mating_system)polygamy  0.232928    246.3 &lt;0.005
                                                      
(Intercept)                                         **
factor(Sex)genCV_male                                 
factor(Mating_system)polygamy                         
factor(Sex)genCV_male:factor(Mating_system)polygamy **
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>And finally, we included the study type (‚ÄòStudyType‚Äô), i.e.¬†if the data were obtained in laboratory or field experiments.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb24-1" title="1">RS_gen_StudyType_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(StudyType),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb24-2" title="2">                                      <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb24-3" title="3">                                      <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb24-4" title="4">                                      <span class="dt">data=</span>RS_gen_metaData,</a>
<a class="sourceLine" id="cb24-5" title="5">                                      <span class="dt">burnin =</span> BURNIN,</a>
<a class="sourceLine" id="cb24-6" title="6">                                      <span class="dt">nitt=</span>NITT,</a>
<a class="sourceLine" id="cb24-7" title="7">                                      <span class="dt">thin=</span>THIN)</a></code></pre></div>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb25-1" title="1"><span class="kw">summary</span>(RS_gen_StudyType_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 10001:109501
 Thinning interval  = 500
 Sample size  = 200 

 DIC: -127.4064 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01346 0.0004328  0.03499      200

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.003754 0.0003163  0.00981    142.8

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID    0.0143 0.003386   0.0273      200

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01555  0.01059  0.02143      200

 Location effects: genCV ~ factor(Sex) * factor(StudyType) 

                                                  post.mean  l-95% CI  u-95% CI
(Intercept)                                        0.171715 -0.088682  0.386666
factor(Sex)genCV_male                              0.080874  0.023493  0.161411
factor(StudyType)Laboratory                        0.127308 -0.071792  0.395470
factor(Sex)genCV_male:factor(StudyType)Laboratory  0.002628 -0.074388  0.084146
                                                  eff.samp pMCMC  
(Intercept)                                          142.8  0.11  
factor(Sex)genCV_male                                258.0  0.04 *
factor(StudyType)Laboratory                          129.2  0.27  
factor(Sex)genCV_male:factor(StudyType)Laboratory    246.0  0.95  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb27-1" title="1"><span class="kw">sessionInfo</span>()</a></code></pre></div>
<pre><code>R version 4.0.0 (2020-04-24)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19041)

Matrix products: default

locale:
[1] LC_COLLATE=German_Germany.1252  LC_CTYPE=German_Germany.1252   
[3] LC_MONETARY=German_Germany.1252 LC_NUMERIC=C                   
[5] LC_TIME=German_Germany.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] readr_1.4.0                ggplot2_3.3.3             
 [3] data.table_1.14.0          stargazer_5.2.2           
 [5] dplyr_1.0.5                MCMCglmm_2.32             
 [7] coda_0.19-4                tidyr_1.1.3               
 [9] PerformanceAnalytics_2.0.4 xts_0.12.1                
[11] zoo_1.8-9                  matrixcalc_1.0-3          
[13] outliers_0.14              psych_2.1.3               
[15] multcomp_1.4-16            TH.data_1.0-10            
[17] survival_3.1-12            mvtnorm_1.1-1             
[19] pwr_1.3-0                  MASS_7.3-51.5             
[21] metafor_2.4-0              Matrix_1.2-18             
[23] ape_5.4-1                  workflowr_1.6.2           

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.6        lattice_0.20-41   corpcor_1.6.9     assertthat_0.2.1 
 [5] rprojroot_2.0.2   digest_0.6.27     utf8_1.2.1        R6_2.5.0         
 [9] evaluate_0.14     highr_0.8         pillar_1.5.1      rlang_0.4.10     
[13] cubature_2.0.4.1  whisker_0.4       rmarkdown_2.7     splines_4.0.0    
[17] stringr_1.4.0     munsell_0.5.0     compiler_4.0.0    httpuv_1.6.1     
[21] xfun_0.22         pkgconfig_2.0.3   mnormt_2.0.2      tmvnsim_1.0-2    
[25] htmltools_0.5.1.1 tidyselect_1.1.0  tibble_3.1.0      tensorA_0.36.2   
[29] quadprog_1.5-8    codetools_0.2-16  fansi_0.4.2       withr_2.4.1      
[33] crayon_1.4.1      later_1.2.0       grid_4.0.0        gtable_0.3.0     
[37] nlme_3.1-147      lifecycle_1.0.0   DBI_1.1.1         git2r_0.28.0     
[41] magrittr_2.0.1    scales_1.1.1      stringi_1.5.3     fs_1.5.0         
[45] promises_1.2.0.1  ellipsis_0.3.1    vctrs_0.3.6       generics_0.1.0   
[49] sandwich_3.0-0    tools_4.0.0       glue_1.4.2        purrr_0.3.4      
[53] hms_1.0.0         parallel_4.0.0    yaml_2.2.1        colorspace_2.0-0 
[57] knitr_1.31       </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
