<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Lennart Winkler1, Maria Moiron2, Edward H. Morrow3 and Tim Janicke1,2   1Applied Zoology, Technical University Dresden   2Centre d’Écologie Fonctionnelle et Évolutive, UMR 5175, CNRS, Université de Montpellier 3Department for Environmental and Life Sciences, Karlstad University" />


<title>Stronger net selection on males across animals</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/yeti.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>


<style type="text/css">
  code {
    white-space: pre;
  }
  .sourceCode {
    overflow: visible;
  }
</style>
<style type="text/css" data-origin="pandoc">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(title);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  { background-color: #f8f8f8; }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #c4a000; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #000000; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #000000; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #000000; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */

.sourceCode .row {
  width: 100%;
}
.sourceCode {
  overflow-x: auto;
}
.code-folding-btn {
  margin-right: -30px;
}
</style>
<script>
// apply pandoc div.sourceCode style to pre.sourceCode instead
(function() {
  var sheets = document.styleSheets;
  for (var i = 0; i < sheets.length; i++) {
    if (sheets[i].ownerNode.dataset["origin"] !== "pandoc") continue;
    try { var rules = sheets[i].cssRules; } catch (e) { continue; }
    for (var j = 0; j < rules.length; j++) {
      var rule = rules[j];
      // check if there is a div.sourceCode rule
      if (rule.type !== rule.STYLE_RULE || rule.selectorText !== "div.sourceCode") continue;
      var style = rule.style.cssText;
      // check if color or background-color is set
      if (rule.style.color === '' && rule.style.backgroundColor === '') continue;
      // replace div.sourceCode by a pre.sourceCode rule
      sheets[i].deleteRule(j);
      sheets[i].insertRule('pre.sourceCode{' + style + '}', j);
    }
  }
})();
</script>







<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-sm-12 col-md-4 col-lg-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-sm-12 col-md-8 col-lg-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Stronger net selection on males across animals</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Analyses</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Stronger net selection on males across animals</h1>
<h3 class="subtitle">Supplementary material reporting R code</h3>
<h4 class="author">Lennart Winkler<sup>1</sup>, Maria Moiron<sup>2</sup>, Edward H. Morrow<sup>3</sup> and Tim Janicke<sup>1,2</sup> <br></br> <sup>1</sup>Applied Zoology, Technical University Dresden <br> <sup>2</sup>Centre d’Écologie Fonctionnelle et Évolutive, UMR 5175, CNRS, Université de Montpellier</br> <sup>3</sup>Department for Environmental and Life Sciences, Karlstad University</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-09-22
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>Net_Selection_eLife_code/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20210613code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20210613)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20210613code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20210613)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomLennartWinklerNetSelectioneLifecodetree8c7305faddbc0995c08d540cb780e447827ae5e9targetblank8c7305fa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/tree/8c7305faddbc0995c08d540cb780e447827ae5e9" target="_blank">8c7305f</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomLennartWinklerNetSelectioneLifecodetree8c7305faddbc0995c08d540cb780e447827ae5e9targetblank8c7305fa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/tree/8c7305faddbc0995c08d540cb780e447827ae5e9" target="_blank">8c7305f</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    data/.Rhistory

Untracked files:
    Untracked:  data/META_SexSpecGenVar_Pylogeny_v05_NEWICK.txt
    Untracked:  data/META_SexSpecGenVar_Pylogeny_v05_PIC.txt
    Untracked:  data/Winkler_et_al_2021_DataTable.txt

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/index.Rmd</code>) and HTML (<code>docs/index.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/dcdf568487731c743f3b59589d222989dcbdbd05/docs/index.html" target="_blank">dcdf568</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-09-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/1d415ea9c8865103a65e6600c74d83b18bc1b67e/analysis/index.Rmd" target="_blank">1d415ea</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-09-22
</td>
<td>
wflow_publish(republish = TRUE, all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/1d415ea9c8865103a65e6600c74d83b18bc1b67e/docs/index.html" target="_blank">1d415ea</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-09-22
</td>
<td>
wflow_publish(republish = TRUE, all = T)
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/f1eb5c9d1a7e6e0715058161c79c0bec610429a8/analysis/index.Rmd" target="_blank">f1eb5c9</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-09-22
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/f1eb5c9d1a7e6e0715058161c79c0bec610429a8/docs/index.html" target="_blank">f1eb5c9</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-09-22
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/0f87db403fbdf7be757730c26a24667d86647764/docs/index.html" target="_blank">0f87db4</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/b46a782b9b174ebc678746971e4e66da01ba67a6/analysis/index.Rmd" target="_blank">b46a782</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/b46a782b9b174ebc678746971e4e66da01ba67a6/docs/index.html" target="_blank">b46a782</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/79ae9878412deacac8585214cf885629ce2fef75/docs/index.html" target="_blank">79ae987</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/f7593bdfbf017206845070ffcd38af59b2e01fb7/analysis/index.Rmd" target="_blank">f7593bd</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/f7593bdfbf017206845070ffcd38af59b2e01fb7/docs/index.html" target="_blank">f7593bd</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/98c553de2cd446b43abfe7ba5a3c6832d45681c0/docs/index.html" target="_blank">98c553d</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/926ceb8061cd359e9440de3e01261a6f4cdc940c/analysis/index.Rmd" target="_blank">926ceb8</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/926ceb8061cd359e9440de3e01261a6f4cdc940c/docs/index.html" target="_blank">926ceb8</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/3bccaab7bbe3abe85ae5e34dce46480154c0301b/docs/index.html" target="_blank">3bccaab</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/7ffcf001b4fa246d15b13673aad1756fd73875a0/analysis/index.Rmd" target="_blank">7ffcf00</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/7ffcf001b4fa246d15b13673aad1756fd73875a0/docs/index.html" target="_blank">7ffcf00</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/63df934dd1971d30dbf57a92db6ba19c8330bd01/docs/index.html" target="_blank">63df934</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/36c6fdc37a65eb88ced0256632c626d756d5383c/analysis/index.Rmd" target="_blank">36c6fdc</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/36c6fdc37a65eb88ced0256632c626d756d5383c/docs/index.html" target="_blank">36c6fdc</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/b3577deff4ea5303a5c13c738e23674634d7ac41/docs/index.html" target="_blank">b3577de</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/85c0b6e8d8dc0026d92e73098cb42bdb8b49d209/analysis/index.Rmd" target="_blank">85c0b6e</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/85c0b6e8d8dc0026d92e73098cb42bdb8b49d209/docs/index.html" target="_blank">85c0b6e</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/4ed93cf9cbeef47676851510b425c05068a55974/docs/index.html" target="_blank">4ed93cf</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/058af7bee1aadb54aced94c34bd68bb541d4fe9c/analysis/index.Rmd" target="_blank">058af7b</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/058af7bee1aadb54aced94c34bd68bb541d4fe9c/docs/index.html" target="_blank">058af7b</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/223df2163ebb54bb9d7b02410a9fcaa9135214a5/docs/index.html" target="_blank">223df21</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/f63b665b10d723a5dd760a11a411f7deecf7d042/analysis/index.Rmd" target="_blank">f63b665</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/f63b665b10d723a5dd760a11a411f7deecf7d042/docs/index.html" target="_blank">f63b665</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-23
</td>
<td>
wflow_publish(all = T)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/3139eb4d54ec90248d7cb998041d07a29bf51d07/docs/index.html" target="_blank">3139eb4</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/9290105671d5938440c5e00147b7b221b92787d1/analysis/index.Rmd" target="_blank">9290105</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-14
</td>
<td>
wflow_publish(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/9290105671d5938440c5e00147b7b221b92787d1/docs/index.html" target="_blank">9290105</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-14
</td>
<td>
wflow_publish(all = TRUE)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/01525f804af4cb282742c6731551ef94df0c4d2c/docs/index.html" target="_blank">01525f8</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/161eee9ca4f35be797a9d11e56fad7e08cdff9b8/analysis/index.Rmd" target="_blank">161eee9</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
wflow_publish(“C:/Users/lenna/Desktop/Documents/PhD/2020_META SexSpecGenVar/eLife/Net_Selection_eLife_code/analysis/index.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/6f60c27281e2554ac17c31a6860265ffc84cc6bb/docs/index.html" target="_blank">6f60c27</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/555a290d73889280bb16552b5911a6335d0e0bc7/analysis/index.Rmd" target="_blank">555a290</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
wflow_publish(“C:/Users/lenna/Documents/Net_Selection_eLife_code/analysis/index.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/9220292c05c265ac4910fef2f664a3db83327342/docs/index.html" target="_blank">9220292</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/15f14fe3c1a44ca94624f55e682db2816b41d055/analysis/index.Rmd" target="_blank">15f14fe</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
wflow_publish(“C:/Users/lenna/Documents/Net_Selection_eLife_code/analysis/index.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/db2f16f4eed457e18792fe29a85af46525caad72/docs/index.html" target="_blank">db2f16f</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/00738afe7fddfe48ff90a166e230dc7ac3615923/analysis/index.Rmd" target="_blank">00738af</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
wflow_publish(“C:/Users/lenna/Documents/Net_Selection_eLife_code/analysis/index.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/LennartWinkler/Net_Selection_eLife_code/7d5ef2c287225924c6124ecb5bccaa216c8a8c56/docs/index.html" target="_blank">7d5ef2c</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/LennartWinkler/Net_Selection_eLife_code/blob/320011ba4a7e729c473d1ce6e756a3bfcfa72046/analysis/index.Rmd" target="_blank">320011b</a>
</td>
<td>
LennartWinkler
</td>
<td>
2021-06-13
</td>
<td>
Start workflowr project.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<p>Supplementary material reporting R code for the manuscript ‘Stronger net selection on males across animals’.</p>
<div id="load-and-prepare-data" class="section level2">
<h2>Load and prepare data</h2>
<p>Before we started the analyses, we loaded all necessary packages and data.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" title="1"><span class="co">#load packages</span></a>
<a class="sourceLine" id="cb1-2" title="2"><span class="kw">rm</span>(<span class="dt">list =</span> <span class="kw">ls</span>())</a>
<a class="sourceLine" id="cb1-3" title="3"><span class="kw">library</span>(ape)</a>
<a class="sourceLine" id="cb1-4" title="4"><span class="kw">library</span>(MCMCglmm)</a>
<a class="sourceLine" id="cb1-5" title="5"><span class="kw">library</span>(caper) <span class="co">#for phylogenetic independent contrasts (PICs)</span></a>
<a class="sourceLine" id="cb1-6" title="6"><span class="kw">library</span>(tidyr)</a>
<a class="sourceLine" id="cb1-7" title="7"><span class="kw">library</span>(matrixcalc)</a>
<a class="sourceLine" id="cb1-8" title="8"><span class="kw">library</span>(lattice)</a>
<a class="sourceLine" id="cb1-9" title="9"><span class="kw">library</span>(ggplot2)</a>
<a class="sourceLine" id="cb1-10" title="10"><span class="kw">library</span>(reshape2)</a>
<a class="sourceLine" id="cb1-11" title="11"></a>
<a class="sourceLine" id="cb1-12" title="12"><span class="co">#load data</span></a>
<a class="sourceLine" id="cb1-13" title="13"><span class="kw">setwd</span>(<span class="st">&quot;.&quot;</span>)</a>
<a class="sourceLine" id="cb1-14" title="14">Data &lt;-<span class="st"> </span><span class="kw">read.delim</span>(<span class="st">&quot;./data/Winkler_et_al_2021_DataTable.txt&quot;</span>)</a>
<a class="sourceLine" id="cb1-15" title="15">theTree &lt;-<span class="st"> </span><span class="kw">read.tree</span>(<span class="st">&quot;./data/META_SexSpecGenVar_Pylogeny_v05_NEWICK.txt&quot;</span>) <span class="co">#phylogenetic tree</span></a>
<a class="sourceLine" id="cb1-16" title="16"><span class="co">#Load phylogenetic tree for PIC analyses</span></a>
<a class="sourceLine" id="cb1-17" title="17">PIC_Tree &lt;-<span class="st"> </span><span class="kw">read.tree</span>(<span class="st">&quot;./data/META_SexSpecGenVar_Pylogeny_v05_PIC.txt&quot;</span>)</a>
<a class="sourceLine" id="cb1-18" title="18"></a>
<a class="sourceLine" id="cb1-19" title="19"><span class="co">## Reorganising and subsetting dataset ####</span></a>
<a class="sourceLine" id="cb1-20" title="20"></a>
<a class="sourceLine" id="cb1-21" title="21">stacked_gen_Data &lt;-<span class="st"> </span><span class="kw">gather</span>(Data, <span class="dt">key =</span> <span class="st">&quot;Sex&quot;</span>,<span class="dt">value =</span> <span class="st">&quot;genCV&quot;</span>, genCV_male, genCV_female)</a>
<a class="sourceLine" id="cb1-22" title="22">stacked_phen_Data &lt;-<span class="st"> </span><span class="kw">gather</span>(Data, <span class="dt">key =</span> <span class="st">&quot;Sex&quot;</span>,<span class="dt">value =</span> <span class="st">&quot;phenCV&quot;</span>, phenCV_male, phenCV_female)</a>
<a class="sourceLine" id="cb1-23" title="23"></a>
<a class="sourceLine" id="cb1-24" title="24">RS_gen_metaData&lt;-<span class="kw">subset</span>(stacked_gen_Data, FitnessCat <span class="op">==</span><span class="st"> &quot;RS&quot;</span>)</a>
<a class="sourceLine" id="cb1-25" title="25">LS_gen_metaData&lt;-<span class="kw">subset</span>(stacked_gen_Data, FitnessCat <span class="op">==</span><span class="st"> &quot;LS&quot;</span>)</a>
<a class="sourceLine" id="cb1-26" title="26"></a>
<a class="sourceLine" id="cb1-27" title="27">RS_phen_metaData&lt;-<span class="kw">subset</span>(stacked_phen_Data, FitnessCat <span class="op">==</span><span class="st"> &quot;RS&quot;</span>)</a>
<a class="sourceLine" id="cb1-28" title="28">LS_phen_metaData&lt;-<span class="kw">subset</span>(stacked_phen_Data, FitnessCat <span class="op">==</span><span class="st"> &quot;LS&quot;</span>)</a>
<a class="sourceLine" id="cb1-29" title="29"></a>
<a class="sourceLine" id="cb1-30" title="30">RS_Data&lt;-<span class="kw">subset</span>(Data, FitnessCat <span class="op">==</span><span class="st"> &quot;RS&quot;</span>)</a>
<a class="sourceLine" id="cb1-31" title="31">LS_Data&lt;-<span class="kw">subset</span>(Data, FitnessCat <span class="op">==</span><span class="st"> &quot;LS&quot;</span>)</a></code></pre></div>
</div>
<div id="phenotypic-gambit" class="section level2">
<h2>Phenotypic gambit</h2>
<p>Statistical analyses were carried out in two steps. First, we examined the key assumption of the ‘phenotypic gambit’ by testing whether estimates of phenotypic variance predict the estimated genetic variance. For this we ran linear regressions with CVP defined as predictor variable and CVG defined as response variable. This was done separately for both sexes and the two fitness components. The analyses on the phenotypic gambit were motivated from a methodological perspective and we did not expect that inter-specific variation in the difference between CVP and CVG can be explained by a shared phylogenetic history. However, for completeness, we also ran linear regressions on phylogenetic independent contrasts (PICs) computed using the crunch function of the caper R-package (version 1.0.1) in R (Orme et al. 2018) to test whether our findings were robust when accounting for potential phylogenetic non-independence.</p>
<p>First, we computed linear regressions between genetic and phenotypic variances, not controlling for the phylogeny. We started with the linear regression in reproductive success for males.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" title="1"><span class="co">### Linear regressions (not controlling for phylogeny) ####</span></a>
<a class="sourceLine" id="cb2-2" title="2"></a>
<a class="sourceLine" id="cb2-3" title="3">LM_Gambit_RS_males   &lt;-<span class="st"> </span><span class="kw">lm</span>(genCV_male   <span class="op">~</span><span class="st"> </span>phenCV_male,   <span class="dt">data =</span> RS_Data) <span class="co">#Reproductive success in males</span></a>
<a class="sourceLine" id="cb2-4" title="4"><span class="kw">summary</span>(LM_Gambit_RS_males) <span class="co">#Reproductive success in males</span></a></code></pre></div>
<pre><code>
Call:
lm(formula = genCV_male ~ phenCV_male, data = RS_Data)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.39325 -0.13953 -0.08111  0.11677  0.62956 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)  0.24747    0.06078   4.071 0.000139 ***
phenCV_male  0.08514    0.05981   1.423 0.159792    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.2214 on 60 degrees of freedom
Multiple R-squared:  0.03267,   Adjusted R-squared:  0.01654 
F-statistic: 2.026 on 1 and 60 DF,  p-value: 0.1598</code></pre>
<p>Then we continued with the linear regression in reproductive success for females.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" title="1">LM_Gambit_RS_females &lt;-<span class="st"> </span><span class="kw">lm</span>(genCV_female <span class="op">~</span><span class="st"> </span>phenCV_female, <span class="dt">data =</span> RS_Data) <span class="co">#Reproductive success in females</span></a>
<a class="sourceLine" id="cb4-2" title="2"><span class="kw">summary</span>(LM_Gambit_RS_females) <span class="co">#Reproductive success in females</span></a></code></pre></div>
<pre><code>
Call:
lm(formula = genCV_female ~ phenCV_female, data = RS_Data)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.24799 -0.10236 -0.02864  0.07889  0.54843 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)    0.21221    0.04865   4.362 5.14e-05 ***
phenCV_female  0.03832    0.06480   0.591    0.556    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.1755 on 60 degrees of freedom
Multiple R-squared:  0.005795,  Adjusted R-squared:  -0.01077 
F-statistic: 0.3497 on 1 and 60 DF,  p-value: 0.5565</code></pre>
<p>Next, the linear regression in lifespan for males.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" title="1">LM_Gambit_LS_males   &lt;-<span class="st"> </span><span class="kw">lm</span>(genCV_male   <span class="op">~</span><span class="st"> </span>phenCV_male,   <span class="dt">data =</span> LS_Data) <span class="co">#Lifespan in males</span></a>
<a class="sourceLine" id="cb6-2" title="2"><span class="kw">summary</span>(LM_Gambit_LS_males) <span class="co">#Lifespan in males</span></a></code></pre></div>
<pre><code>
Call:
lm(formula = genCV_male ~ phenCV_male, data = LS_Data)

Residuals:
      Min        1Q    Median        3Q       Max 
-0.200220 -0.059558  0.006778  0.048454  0.260075 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)  0.04934    0.02969   1.662 0.104960    
phenCV_male  0.26577    0.06794   3.912 0.000378 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.09687 on 37 degrees of freedom
Multiple R-squared:  0.2926,    Adjusted R-squared:  0.2734 
F-statistic:  15.3 on 1 and 37 DF,  p-value: 0.0003778</code></pre>
<p>Finally, the linear regression in lifespan for females.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" title="1">LM_Gambit_LS_females &lt;-<span class="st"> </span><span class="kw">lm</span>(genCV_female <span class="op">~</span><span class="st"> </span>phenCV_female, <span class="dt">data =</span> LS_Data) <span class="co">#Lifespan in females</span></a>
<a class="sourceLine" id="cb8-2" title="2"><span class="kw">summary</span>(LM_Gambit_LS_females) <span class="co">#Lifespan in females</span></a></code></pre></div>
<pre><code>
Call:
lm(formula = genCV_female ~ phenCV_female, data = LS_Data)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.16792 -0.05745  0.01441  0.04686  0.15623 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)    
(Intercept)    0.05455    0.02248   2.426 0.020245 *  
phenCV_female  0.20435    0.05083   4.020 0.000275 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.0733 on 37 degrees of freedom
Multiple R-squared:  0.304, Adjusted R-squared:  0.2852 
F-statistic: 16.16 on 1 and 37 DF,  p-value: 0.0002749</code></pre>
<p>Next, we expanded the previous analyses on the phenotypic gambit, by considering the phylogeny using phylogenetic independent contrasts (PICs).</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" title="1"><span class="co"># Phylogenetic linear regressions using the caper package ####</span></a>
<a class="sourceLine" id="cb10-2" title="2"></a>
<a class="sourceLine" id="cb10-3" title="3"><span class="co">#Build phylogenetic tree for reproductive success</span></a>
<a class="sourceLine" id="cb10-4" title="4">RS_Data<span class="op">$</span>Species_PIC &lt;-<span class="st"> </span><span class="kw">factor</span>(RS_Data<span class="op">$</span>Species_PIC)</a>
<a class="sourceLine" id="cb10-5" title="5">RS_Species_Data_PIC &lt;-<span class="st"> </span><span class="kw">unique</span>(RS_Data<span class="op">$</span>Species_PIC)</a>
<a class="sourceLine" id="cb10-6" title="6">RS_PIC_Tree&lt;-<span class="kw">drop.tip</span>(PIC_Tree, PIC_Tree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(RS_Species_Data_PIC, PIC_Tree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb10-7" title="7"><span class="kw">plot</span>(RS_PIC_Tree)</a></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb11-1" title="1"><span class="co">#Build phylogenetic tree for lifespan</span></a>
<a class="sourceLine" id="cb11-2" title="2">LS_Data<span class="op">$</span>Species_PIC &lt;-<span class="st"> </span><span class="kw">factor</span>(LS_Data<span class="op">$</span>Species_PIC)</a>
<a class="sourceLine" id="cb11-3" title="3">LS_Species_Data_PIC &lt;-<span class="st"> </span><span class="kw">unique</span>(LS_Data<span class="op">$</span>Species_PIC)</a>
<a class="sourceLine" id="cb11-4" title="4">LS_PIC_Tree&lt;-<span class="kw">drop.tip</span>(PIC_Tree, PIC_Tree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(LS_Species_Data_PIC, PIC_Tree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb11-5" title="5"><span class="kw">plot</span>(LS_PIC_Tree)</a></code></pre></div>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" title="1"><span class="co">#Combine phylogeny and data</span></a>
<a class="sourceLine" id="cb12-2" title="2">PGLS_RS_Data    &lt;-<span class="kw">comparative.data</span>(<span class="dt">phy =</span> RS_PIC_Tree, <span class="dt">data =</span> RS_Data, <span class="dt">names.col =</span> Species_PIC, <span class="dt">vcv =</span> <span class="ot">TRUE</span>, <span class="dt">na.omit =</span> <span class="ot">FALSE</span>, <span class="dt">warn.dropped =</span> <span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb12-3" title="3">PGLS_LS_Data    &lt;-<span class="kw">comparative.data</span>(<span class="dt">phy =</span> LS_PIC_Tree, <span class="dt">data =</span> LS_Data, <span class="dt">names.col =</span> Species_PIC, <span class="dt">vcv =</span> <span class="ot">TRUE</span>, <span class="dt">na.omit =</span> <span class="ot">FALSE</span>, <span class="dt">warn.dropped =</span> <span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb12-4" title="4"></a>
<a class="sourceLine" id="cb12-5" title="5"><span class="co">#Run PIC using caper package</span></a>
<a class="sourceLine" id="cb12-6" title="6">PIC_Crunch_mRS &lt;-<span class="st"> </span><span class="kw">crunch</span>(genCV_male   <span class="op">~</span><span class="st"> </span>phenCV_male,   <span class="dt">data=</span>PGLS_RS_Data, <span class="dt">equal.branch.length=</span><span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb12-7" title="7">PIC_Crunch_fRS &lt;-<span class="st"> </span><span class="kw">crunch</span>(genCV_female <span class="op">~</span><span class="st"> </span>phenCV_female, <span class="dt">data=</span>PGLS_RS_Data, <span class="dt">equal.branch.length=</span><span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb12-8" title="8">PIC_Crunch_mLS &lt;-<span class="st"> </span><span class="kw">crunch</span>(genCV_male   <span class="op">~</span><span class="st"> </span>phenCV_male,   <span class="dt">data=</span>PGLS_LS_Data, <span class="dt">equal.branch.length=</span><span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb12-9" title="9">PIC_Crunch_fLS &lt;-<span class="st"> </span><span class="kw">crunch</span>(genCV_female <span class="op">~</span><span class="st"> </span>phenCV_female, <span class="dt">data=</span>PGLS_LS_Data, <span class="dt">equal.branch.length=</span><span class="ot">FALSE</span>)</a>
<a class="sourceLine" id="cb12-10" title="10"></a>
<a class="sourceLine" id="cb12-11" title="11">PIC_Crunch_mRS <span class="co">#PIC male reproductive success</span></a>
<a class="sourceLine" id="cb12-12" title="12">PIC_Crunch_fRS <span class="co">#PIC female reproductive success</span></a>
<a class="sourceLine" id="cb12-13" title="13">PIC_Crunch_mLS <span class="co">#PIC male lifespan</span></a>
<a class="sourceLine" id="cb12-14" title="14">PIC_Crunch_fLS <span class="co">#PIC female lifespan</span></a></code></pre></div>
<p>The results for phylogenetic independent contrast (PIC) for male reproductive success:</p>
<pre><code>Phylogenetic Independent Contrasts analysis using:crunch.
Response values are species rich contrasts using:   

Phylogeny:  (62 tips)
Data:  (62 rows)
Number of valid contrasts: 61

Call:
lm(genCV_male ~ phenCV_male - 1, data = contrData)

Residuals:
    Min      1Q  Median      3Q     Max 
-5.3797 -0.0128  0.0685  1.1351  7.6022 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)  
phenCV_male  0.11328    0.05145   2.202   0.0315 *
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 2.071 on 60 degrees of freedom
Multiple R-squared:  0.07475,   Adjusted R-squared:  0.05933 
F-statistic: 4.847 on 1 and 60 DF,  p-value: 0.03155</code></pre>
<p>The results for phylogenetic independent contrast (PIC) for female reproductive success:</p>
<pre><code>Phylogenetic Independent Contrasts analysis using:crunch.
Response values are species rich contrasts using:   

Phylogeny:  (62 tips)
Data:  (62 rows)
Number of valid contrasts: 61

Call:
lm(genCV_female ~ phenCV_female - 1, data = contrData)

Residuals:
    Min      1Q  Median      3Q     Max 
-4.4166 -0.1539  0.0011  0.5176  3.7100 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)   
phenCV_female  0.19087    0.06503   2.935  0.00472 **
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 1.498 on 60 degrees of freedom
Multiple R-squared:  0.1256,    Adjusted R-squared:  0.111 
F-statistic: 8.615 on 1 and 60 DF,  p-value: 0.00472</code></pre>
<p>The results for phylogenetic independent contrast (PIC) for male lifespan:</p>
<pre><code>Phylogenetic Independent Contrasts analysis using:crunch.
Response values are species rich contrasts using:   

Phylogeny:  (39 tips)
Data:  (39 rows)
Number of valid contrasts: 38

Call:
lm(genCV_male ~ phenCV_male - 1, data = contrData)

Residuals:
     Min       1Q   Median       3Q      Max 
-1.23024 -0.03652 -0.00295  0.24827  1.80770 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)    
phenCV_male  0.55262    0.04826   11.45 1.01e-13 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.5219 on 37 degrees of freedom
Multiple R-squared:  0.7799,    Adjusted R-squared:  0.774 
F-statistic: 131.1 on 1 and 37 DF,  p-value: 1.008e-13</code></pre>
<p>The results for phylogenetic independent contrast (PIC) for female lifespan:</p>
<pre><code>Phylogenetic Independent Contrasts analysis using:crunch.
Response values are species rich contrasts using:   

Phylogeny:  (39 tips)
Data:  (39 rows)
Number of valid contrasts: 38

Call:
lm(genCV_female ~ phenCV_female - 1, data = contrData)

Residuals:
     Min       1Q   Median       3Q      Max 
-1.19148 -0.01920 -0.00039  0.31029  1.23392 

Coefficients:
              Estimate Std. Error t value Pr(&gt;|t|)    
phenCV_female  0.34523    0.03616   9.547  1.6e-11 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.4447 on 37 degrees of freedom
Multiple R-squared:  0.7112,    Adjusted R-squared:  0.7034 
F-statistic: 91.14 on 1 and 37 DF,  p-value: 1.6e-11</code></pre>
</div>
<div id="stronger-net-selection-on-males-pglmms" class="section level2">
<h2>Stronger net selection on males (PGLMMs)</h2>
<p>In this second part of the analysis, we tested the hypothesis that net selection is stronger on males by testing for a male bias in CVP and CVG. Specifically, we ran Phylogenetic General Linear Mixed-Effects Models (PGLMMs) with CVP or CVG as the response variable, and sex as a fixed effect. To account for the paired data structure, we added an observation identifier as a random effect. Moreover, all models included a study identifier and the phylogeny (transformed into a correlation matrix) as random effects to account for statistical non-independence arising from shared study design or phylogenetic history, respectively. Note that the latter also accounts for the non-independence of estimates obtained from the same species as some studies estimated genetic variances from distinct field populations (Fox et al. 2004) or different experimental treatments under laboratory conditions such as food stress (Holman &amp; Jacomb 2017) and temperature stress (Berger et al. 2014). In an additional series of PGLMMs we tested whether our proxy of sexual selection explained inter-specific variation in the sex-differences of CVP or CVG by adding mating system and its interaction with sex as fixed effects to the models. These analyses were run on both the complete dataset and on a subset including only vertebrates. The latter set of analyses were done to acknowledge the fact that all sampled invertebrate species were classified as polygamous and the three independent evolutionary events in our phylogeny marking transitions between mating systems occurred in vertebrates (including 6 polygamous species, 6 monogamous species). Finally, given that primary studies varied in several methodological aspects (see section ‘Literature Search and Characterization of Primary Studies’) and whenever the level of replication allowed statistical analysis, we tested whether the different approaches predicted estimates of CVP and CVG and affected their sex-difference. Specifically, we ran three PGLMMs to test separately for effects of the study type, the estimate of genetic variance VG, and the type of reproductive success metric (see above). We carried out PGLMMs with the MCMCglmm R-package (version 2.2.9) (Hadfield 2010), using uninformative priors (V = 1, nu = 0.01) and an effective sample size of 20000 (number of iterations = 11000000, burn-in = 1000000, thinning interval = 500). We computed the proportion of variance explained by fixed factors (‘marginal R2’) (Nakagawa &amp; Schielzeth 2013). In addition, we quantified the phylogenetic signal as the phylogenetic heritability H2 (i.e., proportional variance in CVP or CVG explained by the phylogeny) (de Villemereuil &amp; Nakagawa 2014). In a previous study testing for sex-specific phenotypic variances in reproductive success (Janicke et al. 2016), we ran formal meta-analyses using lnCVR as the tested effect size (Nakagawa et al. 2015). This is potentially a more powerful approach for comparing phenotypic variances but rendered unsuitable when comparing genetic variances. This is because the computation of the sampling variance of lnCVR is a function of the sample size of the sampled population and the point estimate of lnCVR (Nakagawa et al. 2015). However, genetic variances are estimates from statistical models and notorious for being estimated with low precision (i.e., have large confidence intervals). Therefore, using a meta-analytic approach for genetic variances using lnCVR as an effect size leads to overconfident estimation of the global effect size and is therefore likely to result in type-II-errors. However, to allow comparison with the previous meta-analysis, we report the outcome of phylogenetic meta-analyses on phenotypic variances using lnCVR in the Supplementary Material (Table S3), which largely reflects the results on the point estimates of CVP from PGLMMs.</p>
<div id="run-mcmc-models" class="section level3">
<h3>Run MCMC models</h3>
<p>We then ran MCMC models to test for differences in the phenotypic and genetic variances in males and females for the fitness categories reproductive success and lifespan.</p>
<p>First, we pruned the phylogenetic tree to the data subset.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb17-1" title="1"><span class="co">## PRUNE PHYLOGENETIC TREE TO DATA SUBSET</span></a>
<a class="sourceLine" id="cb17-2" title="2">RS_gen_metaData<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(RS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb17-3" title="3"><span class="kw">is.factor</span>(RS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb17-4" title="4">RS_Species_Data &lt;-<span class="st"> </span><span class="kw">unique</span>(RS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb17-5" title="5"><span class="kw">summary</span>(RS_Species_Data)</a>
<a class="sourceLine" id="cb17-6" title="6">RS_theTree&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(RS_Species_Data, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb17-7" title="7"><span class="kw">plot</span>(RS_theTree)</a></code></pre></div>
<p><img src="figure/index.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>We then checked if the phylogenetic tree had been build correctly.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb18-1" title="1"><span class="co">## CHECK PHYLOGENETIC TREE</span></a>
<a class="sourceLine" id="cb18-2" title="2"><span class="kw">sort</span>(RS_theTree<span class="op">$</span>tip.label) <span class="op">==</span><span class="st"> </span><span class="kw">sort</span>(<span class="kw">unique</span>(RS_gen_metaData<span class="op">$</span>animal)) <span class="co"># check if tip names correspond to data names</span></a>
<a class="sourceLine" id="cb18-3" title="3"><span class="kw">is.ultrametric</span>(RS_theTree) <span class="co"># check if BL are aligned contemporaneously</span></a>
<a class="sourceLine" id="cb18-4" title="4"><span class="kw">isSymmetric</span>(<span class="kw">vcv</span>(RS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)) <span class="co"># check symmetry of phylogenetic correlation matrix</span></a>
<a class="sourceLine" id="cb18-5" title="5">rawC &lt;-<span class="st"> </span><span class="kw">vcv</span>(RS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb18-6" title="6"><span class="kw">is.positive.definite</span>(rawC) <span class="co"># if FALSE will have to force symmetry</span></a>
<a class="sourceLine" id="cb18-7" title="7">forcedC &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(<span class="kw">forceSymmetric</span>(<span class="kw">vcv</span>(RS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)))</a>
<a class="sourceLine" id="cb18-8" title="8"><span class="kw">is.positive.definite</span>(forcedC)</a>
<a class="sourceLine" id="cb18-9" title="9">comparedC &lt;-<span class="st"> </span>rawC <span class="op">==</span><span class="st"> </span>forcedC</a>
<a class="sourceLine" id="cb18-10" title="10">rawC[<span class="kw">cbind</span>(<span class="kw">which</span>(comparedC<span class="op">!=</span><span class="ot">TRUE</span>, <span class="dt">arr.ind =</span> T))] <span class="op">-</span><span class="st"> </span>forcedC[<span class="kw">cbind</span>(<span class="kw">which</span>(comparedC<span class="op">!=</span><span class="ot">TRUE</span>, <span class="dt">arr.ind =</span> T))] <span class="op">&lt;</span><span class="st"> </span><span class="fl">1e-5</span></a></code></pre></div>
<p>Next, we set the priors for the MCMC models using uninformative priors (V = 1, nu = 0.01) and an effective sample size of 20000 (number of iterations = 11000000, burn-in = 1000000, thinning interval = 500). We also set the index and study ID as factors.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb19-1" title="1"><span class="co">## Prior settings, iterations</span></a>
<a class="sourceLine" id="cb19-2" title="2">pr&lt;-<span class="kw">list</span>(<span class="dt">R=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.01</span>),  <span class="dt">G=</span><span class="kw">list</span>(<span class="dt">G1=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.01</span>),</a>
<a class="sourceLine" id="cb19-3" title="3">                                      <span class="dt">G2=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.01</span>),</a>
<a class="sourceLine" id="cb19-4" title="4">                                      <span class="dt">G3=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.01</span>)))</a>
<a class="sourceLine" id="cb19-5" title="5"></a>
<a class="sourceLine" id="cb19-6" title="6"></a>
<a class="sourceLine" id="cb19-7" title="7">BURNIN =<span class="st">  </span><span class="dv">1000000</span></a>
<a class="sourceLine" id="cb19-8" title="8">NITT   =<span class="st"> </span><span class="dv">11000000</span></a>
<a class="sourceLine" id="cb19-9" title="9">THIN   =<span class="st">      </span><span class="dv">500</span></a>
<a class="sourceLine" id="cb19-10" title="10"></a>
<a class="sourceLine" id="cb19-11" title="11"></a>
<a class="sourceLine" id="cb19-12" title="12">RS_phen_metaData<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_phen_metaData<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb19-13" title="13">RS_phen_metaData<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_phen_metaData<span class="op">$</span>Study_ID)</a>
<a class="sourceLine" id="cb19-14" title="14">RS_gen_metaData<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_gen_metaData<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb19-15" title="15">RS_gen_metaData<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_gen_metaData<span class="op">$</span>Study_ID)</a></code></pre></div>
<div id="mcmc-models-for-reproductive-success" class="section level4">
<h4>MCMC models for reproductive success</h4>
<p>First, we ran the MCMC testing for overall differences in phenotypic variance in reproductive success (‘phenCV’) between males and females (‘Sex’). The model includes the species (‘animal’), estimate ID (‘Index’) and study ID (‘Study_ID’) as random factors (Table 1).</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb20-1" title="1">RS_phen_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb20-2" title="2">                          <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb20-3" title="3">                          <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb20-4" title="4">                          <span class="dt">data=</span>RS_phen_metaData,</a>
<a class="sourceLine" id="cb20-5" title="5">                          <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb20-6" title="6"></a>
<a class="sourceLine" id="cb20-7" title="7"><span class="kw">summary</span>(RS_phen_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: 49.23811 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal   0.03025 0.001009  0.09079    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.05891 0.004482   0.1198    20127

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID    0.0629 0.0009535   0.1457    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.05958  0.04009  0.08303    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC    
(Intercept)               0.6940   0.4801   0.9185    20000  2e-04 ***
factor(Sex)phenCV_male    0.2341   0.1502   0.3231    20000 &lt;5e-05 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>We then expanded the model to include an interaction of mating system (‘Mating_system’) and sex (Table S4).</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb22-1" title="1">RS_phen_MatSyst_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb22-2" title="2">                                  <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb22-3" title="3">                                  <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb22-4" title="4">                                  <span class="dt">data=</span>RS_phen_metaData,</a>
<a class="sourceLine" id="cb22-5" title="5">                                  <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb22-6" title="6"></a>
<a class="sourceLine" id="cb22-7" title="7"><span class="kw">summary</span>(RS_phen_MatSyst_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: 30.63378 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal   0.04896 0.001384   0.1414    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.06211 0.007444   0.1213    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.05915 0.001256   0.1407    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.05011  0.03313  0.06948    20000

 Location effects: phenCV ~ factor(Sex) * factor(Mating_system) 

                                                     post.mean l-95% CI
(Intercept)                                            0.67560  0.28295
factor(Sex)phenCV_male                                -0.01151 -0.17385
factor(Mating_system)polygamy                          0.02515 -0.34242
factor(Sex)phenCV_male:factor(Mating_system)polygamy   0.32320  0.13743
                                                     u-95% CI eff.samp  pMCMC
(Intercept)                                           1.04732    19513 0.0044
factor(Sex)phenCV_male                                0.14407    20698 0.8854
factor(Mating_system)polygamy                         0.40737    20000 0.9133
factor(Sex)phenCV_male:factor(Mating_system)polygamy  0.50448    20433 0.0005
                                                        
(Intercept)                                          ** 
factor(Sex)phenCV_male                                  
factor(Mating_system)polygamy                           
factor(Sex)phenCV_male:factor(Mating_system)polygamy ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC models examining the genetic variance in reproductive success (‘genCV’). First, the overall model for a sex difference (Table 1).</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb24-1" title="1">RS_gen_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb24-2" title="2">                       <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb24-3" title="3">                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb24-4" title="4">                       <span class="dt">data=</span>RS_gen_metaData,</a>
<a class="sourceLine" id="cb24-5" title="5">                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb24-6" title="6"><span class="kw">summary</span>(RS_gen_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -131.1002 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal   0.01675 0.000811  0.04572    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index  0.005744 0.001154  0.01197    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID    0.0143 0.004079  0.02622    19803

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01469  0.01008  0.01966    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp  pMCMC    
(Intercept)             0.25736  0.11555  0.40998    20000 0.0015 ** 
factor(Sex)genCV_male   0.08653  0.04389  0.12967    20000 0.0002 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Secondly, we included an interaction of mating system (‘Mating_system’) and sex (Table S4).</p>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb26-1" title="1">RS_gen_MatSyst_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb26-2" title="2">                  <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb26-3" title="3">                  <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb26-4" title="4">                  <span class="dt">data=</span>RS_gen_metaData,</a>
<a class="sourceLine" id="cb26-5" title="5">                  <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb26-6" title="6"><span class="kw">summary</span>(RS_gen_MatSyst_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -140.0636 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01649 0.0009349   0.0464    20108

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.00612 0.001243  0.01246    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01475 0.004272  0.02715    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01336 0.009143  0.01815    20000

 Location effects: genCV ~ factor(Sex) * factor(Mating_system) 

                                                    post.mean l-95% CI u-95% CI
(Intercept)                                           0.26415  0.08402  0.46431
factor(Sex)genCV_male                                -0.01501 -0.09852  0.06915
factor(Mating_system)polygamy                        -0.01125 -0.16752  0.14601
factor(Sex)genCV_male:factor(Mating_system)polygamy   0.13382  0.03913  0.23038
                                                    eff.samp  pMCMC   
(Intercept)                                            20000 0.0048 **
factor(Sex)genCV_male                                  20000 0.7240   
factor(Mating_system)polygamy                          20000 0.8884   
factor(Sex)genCV_male:factor(Mating_system)polygamy    20353 0.0084 **
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
<div id="mcmc-models-for-lifespan" class="section level4">
<h4>MCMC models for lifespan</h4>
<p>In addition to the previous models on reproductive success, ran MCMC models to test for differences in the phenotypic and genetic variances in males and females for lifespan.</p>
<p>First, we pruned the phylogenetic tree to the data subset.</p>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb28-1" title="1"><span class="co">## PRUNE PHYLOGENETIC TREE TO DATA SUBSET</span></a>
<a class="sourceLine" id="cb28-2" title="2">LS_gen_metaData<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(LS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb28-3" title="3"><span class="kw">is.factor</span>(LS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb28-4" title="4">LS_Species_Data &lt;-<span class="st"> </span><span class="kw">unique</span>(LS_gen_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb28-5" title="5"><span class="kw">summary</span>(LS_Species_Data)</a>
<a class="sourceLine" id="cb28-6" title="6">LS_theTree&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(LS_Species_Data, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb28-7" title="7"><span class="kw">plot</span>(LS_theTree)</a></code></pre></div>
<p><img src="figure/index.Rmd/unnamed-chunk-22-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>We then checked if the phylogenetic tree had been build correctly.</p>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb29-1" title="1"><span class="co">## CHECK PHYLOGENETIC TREE</span></a>
<a class="sourceLine" id="cb29-2" title="2"><span class="kw">sort</span>(LS_theTree<span class="op">$</span>tip.label) <span class="op">==</span><span class="st"> </span><span class="kw">sort</span>(<span class="kw">unique</span>(LS_gen_metaData<span class="op">$</span>animal)) <span class="co"># check if tip names correspond to data names</span></a>
<a class="sourceLine" id="cb29-3" title="3"><span class="kw">is.ultrametric</span>(LS_theTree) <span class="co"># check if BL are aligned contemporaneously</span></a>
<a class="sourceLine" id="cb29-4" title="4"><span class="kw">isSymmetric</span>(<span class="kw">vcv</span>(LS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)) <span class="co"># check symmetry of phylogenetic correlation matrix</span></a>
<a class="sourceLine" id="cb29-5" title="5">rawC &lt;-<span class="st"> </span><span class="kw">vcv</span>(LS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)</a>
<a class="sourceLine" id="cb29-6" title="6"><span class="kw">is.positive.definite</span>(rawC) <span class="co"># if FALSE will have to force symmetry</span></a>
<a class="sourceLine" id="cb29-7" title="7">forcedC &lt;-<span class="st"> </span><span class="kw">as.matrix</span>(<span class="kw">forceSymmetric</span>(<span class="kw">vcv</span>(LS_theTree, <span class="dt">corr=</span><span class="ot">TRUE</span>)))</a>
<a class="sourceLine" id="cb29-8" title="8"><span class="kw">is.positive.definite</span>(forcedC)</a>
<a class="sourceLine" id="cb29-9" title="9">comparedC &lt;-<span class="st"> </span>rawC <span class="op">==</span><span class="st"> </span>forcedC</a>
<a class="sourceLine" id="cb29-10" title="10">rawC[<span class="kw">cbind</span>(<span class="kw">which</span>(comparedC<span class="op">!=</span><span class="ot">TRUE</span>, <span class="dt">arr.ind =</span> T))] <span class="op">-</span><span class="st"> </span>forcedC[<span class="kw">cbind</span>(<span class="kw">which</span>(comparedC<span class="op">!=</span><span class="ot">TRUE</span>, <span class="dt">arr.ind =</span> T))] <span class="op">&lt;</span><span class="st"> </span><span class="fl">1e-5</span></a></code></pre></div>
<p>Next, we set the priors for the MCMC models using uninformative priors (V = 1, nu = 0.002) and an effective sample size of 20000 (number of iterations = 11000000, burn-in = 1000000, thinning interval = 500). We also set the index and study ID as factors.</p>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb30-1" title="1"><span class="co">## Prior settings, iterations</span></a>
<a class="sourceLine" id="cb30-2" title="2">pr&lt;-<span class="kw">list</span>(<span class="dt">R=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.01</span>), <span class="dt">G=</span><span class="kw">list</span>(<span class="dt">G1=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.01</span>),</a>
<a class="sourceLine" id="cb30-3" title="3">                                      <span class="dt">G2=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.01</span>),</a>
<a class="sourceLine" id="cb30-4" title="4">                                      <span class="dt">G3=</span><span class="kw">list</span>(<span class="dt">V=</span><span class="dv">1</span>,<span class="dt">nu=</span><span class="fl">0.01</span>)))</a>
<a class="sourceLine" id="cb30-5" title="5"></a>
<a class="sourceLine" id="cb30-6" title="6">BURNIN =<span class="st">  </span><span class="dv">1000000</span></a>
<a class="sourceLine" id="cb30-7" title="7">NITT   =<span class="st"> </span><span class="dv">11000000</span></a>
<a class="sourceLine" id="cb30-8" title="8">THIN   =<span class="st">      </span><span class="dv">500</span></a>
<a class="sourceLine" id="cb30-9" title="9"></a>
<a class="sourceLine" id="cb30-10" title="10"></a>
<a class="sourceLine" id="cb30-11" title="11">LS_phen_metaData<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(LS_phen_metaData<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb30-12" title="12">LS_phen_metaData<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(LS_phen_metaData<span class="op">$</span>Study_ID)</a>
<a class="sourceLine" id="cb30-13" title="13">LS_gen_metaData<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(LS_gen_metaData<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb30-14" title="14">LS_gen_metaData<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(LS_gen_metaData<span class="op">$</span>Study_ID)</a></code></pre></div>
<p>First, we ran the MCMC testing for overall differences in phenotypic variance in lifespan (‘phenCV’) between males and females (‘Sex’). The model includes the species (‘animal’), estimate ID (‘Index’) and study ID (‘Study_ID’) as random factors (Table 1).</p>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb31-1" title="1">LS_phen_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb31-2" title="2">                               <span class="dt">pedigree=</span>LS_theTree,</a>
<a class="sourceLine" id="cb31-3" title="3">                               <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb31-4" title="4">                               <span class="dt">data=</span>LS_phen_metaData,</a>
<a class="sourceLine" id="cb31-5" title="5">                               <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb31-6" title="6"></a>
<a class="sourceLine" id="cb31-7" title="7"><span class="kw">summary</span>(LS_phen_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -189.6669 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal   0.04176 0.001415   0.1304    18236

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.01119 0.002609  0.02341    20620

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.02779 0.005106  0.05456    19469

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.00341  0.00201 0.005072    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean  l-95% CI  u-95% CI eff.samp  pMCMC   
(Intercept)             0.430756  0.175653  0.689118    20000 0.0073 **
factor(Sex)phenCV_male -0.004789 -0.030274  0.021496    20000 0.7104   
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>We then expanded the model to include an interaction of mating system (‘Mating_system’) and sex (Table S4).</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb33-1" title="1">LS_phen_MatSyst_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb33-2" title="2">                                       <span class="dt">pedigree=</span>LS_theTree,</a>
<a class="sourceLine" id="cb33-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb33-4" title="4">                                       <span class="dt">data=</span>LS_phen_metaData,</a>
<a class="sourceLine" id="cb33-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb33-6" title="6"></a>
<a class="sourceLine" id="cb33-7" title="7"><span class="kw">summary</span>(LS_phen_MatSyst_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -190.3721 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.06899 0.0007289   0.2345    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.01096 0.002546   0.0227    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.02674 0.003981  0.05366    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.00334 0.001979 0.005002    20000

 Location effects: phenCV ~ factor(Sex) * factor(Mating_system) 

                                                     post.mean l-95% CI
(Intercept)                                            0.35987 -0.04926
factor(Sex)phenCV_male                                 0.02678 -0.02719
factor(Mating_system)polygamy                          0.10169 -0.22019
factor(Sex)phenCV_male:factor(Mating_system)polygamy  -0.04085 -0.10084
                                                     u-95% CI eff.samp  pMCMC  
(Intercept)                                           0.72282    20000 0.0871 .
factor(Sex)phenCV_male                                0.07867    19197 0.3186  
factor(Mating_system)polygamy                         0.43592    20000 0.5719  
factor(Sex)phenCV_male:factor(Mating_system)polygamy  0.02066    19512 0.1834  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC models examining the genetic variance in lifespan (‘genCV’). First, the overall model for a sex difference (Table 1).</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb35-1" title="1">LS_gen_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb35-2" title="2">                              <span class="dt">pedigree=</span>LS_theTree,</a>
<a class="sourceLine" id="cb35-3" title="3">                              <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb35-4" title="4">                              <span class="dt">data=</span>LS_gen_metaData,</a>
<a class="sourceLine" id="cb35-5" title="5">                              <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb35-6" title="6"><span class="kw">summary</span>(LS_gen_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -214.4485 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01277 0.0009457   0.0359    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.002128 0.0006209  0.00431    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.00633 0.001698  0.01227    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.002644 0.001612  0.00378    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean  l-95% CI  u-95% CI eff.samp  pMCMC  
(Intercept)            0.140077 -0.002497  0.281642    20542 0.0529 .
factor(Sex)genCV_male  0.016722 -0.005430  0.040323    20000 0.1464  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Secondly, including an interaction of mating system (‘Mating_system’) and sex (Table S4).</p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb37-1" title="1">LS_gen_MatSyst_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb37-2" title="2">                                      <span class="dt">pedigree=</span>LS_theTree,</a>
<a class="sourceLine" id="cb37-3" title="3">                                      <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb37-4" title="4">                                      <span class="dt">data=</span>LS_gen_metaData,</a>
<a class="sourceLine" id="cb37-5" title="5">                                      <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb37-6" title="6"><span class="kw">summary</span>(LS_gen_MatSyst_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -216.0559 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01274 0.0009163  0.03568    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.00212 0.0005475  0.00422    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID  0.006334 0.001703   0.0122    19491

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.00256 0.001542 0.003665    20000

 Location effects: genCV ~ factor(Sex) * factor(Mating_system) 

                                                    post.mean  l-95% CI
(Intercept)                                          0.162764  0.001801
factor(Sex)genCV_male                                0.049759  0.003971
factor(Mating_system)polygamy                       -0.035428 -0.162993
factor(Sex)genCV_male:factor(Mating_system)polygamy -0.042970 -0.095263
                                                     u-95% CI eff.samp  pMCMC  
(Intercept)                                          0.329991    20000 0.0538 .
factor(Sex)genCV_male                                0.097362    19913 0.0391 *
factor(Mating_system)polygamy                        0.098228    20000 0.5792  
factor(Sex)genCV_male:factor(Mating_system)polygamy  0.011890    19298 0.1159  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
</div>
<div id="subsetting-data-by-mating-system" class="section level3">
<h3>Subsetting data by mating system</h3>
<p>We then analysed the data separately for socially monogamous and polygamous species.</p>
<p>First we subset the data into socially monogamous and polygamous species for all fitness categories (lifespan and reproductive success) and their phenotypic as well as genetic variances.</p>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb39-1" title="1"><span class="co">## Additional subsetting</span></a>
<a class="sourceLine" id="cb39-2" title="2">RS_gen_Monogamy_metaData&lt;-<span class="kw">subset</span>(RS_gen_metaData, Mating_system <span class="op">==</span><span class="st"> &quot;monogamy&quot;</span>)</a>
<a class="sourceLine" id="cb39-3" title="3">RS_gen_Polygamy_metaData&lt;-<span class="kw">subset</span>(RS_gen_metaData, Mating_system <span class="op">==</span><span class="st"> &quot;polygamy&quot;</span>)</a>
<a class="sourceLine" id="cb39-4" title="4"></a>
<a class="sourceLine" id="cb39-5" title="5">RS_phen_Monogamy_metaData&lt;-<span class="kw">subset</span>(RS_phen_metaData, Mating_system <span class="op">==</span><span class="st"> &quot;monogamy&quot;</span>)</a>
<a class="sourceLine" id="cb39-6" title="6">RS_phen_Polygamy_metaData&lt;-<span class="kw">subset</span>(RS_phen_metaData, Mating_system <span class="op">==</span><span class="st"> &quot;polygamy&quot;</span>)</a>
<a class="sourceLine" id="cb39-7" title="7"></a>
<a class="sourceLine" id="cb39-8" title="8">LS_gen_Monogamy_metaData&lt;-<span class="kw">subset</span>(LS_gen_metaData, Mating_system <span class="op">==</span><span class="st"> &quot;monogamy&quot;</span>)</a>
<a class="sourceLine" id="cb39-9" title="9">LS_gen_Polygamy_metaData&lt;-<span class="kw">subset</span>(LS_gen_metaData, Mating_system <span class="op">==</span><span class="st"> &quot;polygamy&quot;</span>)</a>
<a class="sourceLine" id="cb39-10" title="10"></a>
<a class="sourceLine" id="cb39-11" title="11">LS_phen_Monogamy_metaData&lt;-<span class="kw">subset</span>(LS_phen_metaData, Mating_system <span class="op">==</span><span class="st"> &quot;monogamy&quot;</span>)</a>
<a class="sourceLine" id="cb39-12" title="12">LS_phen_Polygamy_metaData&lt;-<span class="kw">subset</span>(LS_phen_metaData, Mating_system <span class="op">==</span><span class="st"> &quot;polygamy&quot;</span>)</a></code></pre></div>
<p>We then pruned the phylogenetic tree to all species with data on lifespan or reproductive success for monogamous and polygamous species separately.</p>
<div class="sourceCode" id="cb40"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb40-1" title="1"><span class="co">## PRUNE PHYLOGENETIC TREES</span></a>
<a class="sourceLine" id="cb40-2" title="2">RS_gen_Monogamy_metaData<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(RS_gen_Monogamy_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb40-3" title="3">RS_Monogamy_Species_Data &lt;-<span class="st"> </span><span class="kw">unique</span>(RS_gen_Monogamy_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb40-4" title="4"><span class="kw">summary</span>(RS_Monogamy_Species_Data)</a>
<a class="sourceLine" id="cb40-5" title="5">RS_Monogamy_theTree&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(RS_Monogamy_Species_Data, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb40-6" title="6"><span class="kw">plot</span>(RS_Monogamy_theTree)</a></code></pre></div>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb41-1" title="1">RS_gen_Polygamy_metaData<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(RS_gen_Polygamy_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb41-2" title="2">RS_Polygamy_Species_Data &lt;-<span class="st"> </span><span class="kw">unique</span>(RS_gen_Polygamy_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb41-3" title="3"><span class="kw">summary</span>(RS_Polygamy_Species_Data)</a>
<a class="sourceLine" id="cb41-4" title="4">RS_Polygamy_theTree&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(RS_Polygamy_Species_Data, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb41-5" title="5"><span class="kw">plot</span>(RS_Polygamy_theTree)</a></code></pre></div>
<div class="sourceCode" id="cb42"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb42-1" title="1">LS_gen_Monogamy_metaData<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(LS_gen_Monogamy_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb42-2" title="2">LS_Monogamy_Species_Data &lt;-<span class="st"> </span><span class="kw">unique</span>(LS_gen_Monogamy_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb42-3" title="3"><span class="kw">summary</span>(LS_Monogamy_Species_Data)</a>
<a class="sourceLine" id="cb42-4" title="4">LS_Monogamy_theTree&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(LS_Monogamy_Species_Data, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb42-5" title="5"><span class="kw">plot</span>(LS_Monogamy_theTree)</a></code></pre></div>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb43-1" title="1">LS_gen_Polygamy_metaData<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(LS_gen_Polygamy_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb43-2" title="2">LS_Polygamy_Species_Data &lt;-<span class="st"> </span><span class="kw">unique</span>(LS_gen_Polygamy_metaData<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb43-3" title="3"><span class="kw">summary</span>(LS_Polygamy_Species_Data)</a>
<a class="sourceLine" id="cb43-4" title="4">LS_Polygamy_theTree&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(LS_Polygamy_Species_Data, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb43-5" title="5"><span class="kw">plot</span>(LS_Polygamy_theTree)</a></code></pre></div>
<div id="mcmcs-for-reproductive-success" class="section level4">
<h4>MCMCs for reproductive success</h4>
<p>We then ran the MCMC models, first for the phenotypic variance in reproductive success in monogamous species (Table 1).</p>
<div class="sourceCode" id="cb44"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb44-1" title="1">RS_phen_Monogamy_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb44-2" title="2">                               <span class="dt">pedigree=</span>RS_Monogamy_theTree,</a>
<a class="sourceLine" id="cb44-3" title="3">                               <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb44-4" title="4">                               <span class="dt">data=</span>RS_phen_Monogamy_metaData,</a>
<a class="sourceLine" id="cb44-5" title="5">                               <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb44-6" title="6"><span class="kw">summary</span>(RS_phen_Monogamy_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -65.819 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.5285 0.0008104    2.457    20466

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.09383 0.0008326   0.2588    19526

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.09331 0.0009878   0.2594    20310

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.004682 0.001673 0.008786    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC  
(Intercept)              0.85145 -0.11209  1.84827    19853 0.0696 .
factor(Sex)phenCV_male  -0.01228 -0.06019  0.03900    20000 0.5982  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC model for the genetic variance in reproductive success in monogamous species (Table 1).</p>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb46-1" title="1">RS_gen_Monogamy_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb46-2" title="2">                                        <span class="dt">pedigree=</span>RS_Monogamy_theTree,</a>
<a class="sourceLine" id="cb46-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb46-4" title="4">                                        <span class="dt">data=</span>RS_gen_Monogamy_metaData,</a>
<a class="sourceLine" id="cb46-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb46-6" title="6"><span class="kw">summary</span>(RS_gen_Monogamy_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -51.50387 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.0603 0.0006262   0.2358    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.01264 0.001037  0.03208    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.01264 0.0009578  0.03221    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.007524 0.002912  0.01365    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp pMCMC
(Intercept)             0.21237 -0.13103  0.53680    20533 0.125
factor(Sex)genCV_male  -0.01467 -0.07633  0.04813    20000 0.624</code></pre>
<p>Then followed the models for reproductive success in polygamous species. First, the MCMC for the phenotypic variance in reproductive success in polygamous species (Table 1).</p>
<div class="sourceCode" id="cb48"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb48-1" title="1">RS_phen_Polygamy_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb48-2" title="2">                                        <span class="dt">pedigree=</span>RS_Polygamy_theTree,</a>
<a class="sourceLine" id="cb48-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb48-4" title="4">                                        <span class="dt">data=</span>RS_phen_Polygamy_metaData,</a>
<a class="sourceLine" id="cb48-5" title="5">                                        <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb48-6" title="6"><span class="kw">summary</span>(RS_phen_Polygamy_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: 43.71919 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.05677 0.0009819   0.1682    20561

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.05064 0.002271   0.1065    19507

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.05453 0.001008    0.142    20293

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.06591  0.04087  0.09548    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC    
(Intercept)               0.7011   0.4155   0.9878    20000  4e-04 ***
factor(Sex)phenCV_male    0.3121   0.2116   0.4210    20000 &lt;5e-05 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC model for the genetic variance in reproductive success in polygamous species (Table 1).</p>
<div class="sourceCode" id="cb50"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb50-1" title="1">RS_gen_Polygamy_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb50-2" title="2">                                       <span class="dt">pedigree=</span>RS_Polygamy_theTree,</a>
<a class="sourceLine" id="cb50-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb50-4" title="4">                                       <span class="dt">data=</span>RS_gen_Polygamy_metaData,</a>
<a class="sourceLine" id="cb50-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb50-6" title="6"><span class="kw">summary</span>(RS_gen_Polygamy_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -95.4313 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01739 0.0008811   0.0504    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index  0.006173 0.001135  0.01314    19406

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01805 0.004008  0.03502    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01545 0.009833  0.02132    19382

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp  pMCMC    
(Intercept)             0.25194  0.09535  0.41043    20000 0.0053 ** 
factor(Sex)genCV_male   0.11857  0.06907  0.16870    20000 &lt;5e-05 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
<div id="mcmcs-for-lifespan" class="section level4">
<h4>MCMCs for lifespan</h4>
<p>Now followed the mating system specific MCMCs for lifespan. We first ran the MCMC models for the phenotypic variance in lifespan in monogamous species (Table 1).</p>
<div class="sourceCode" id="cb52"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb52-1" title="1">LS_phen_Monogamy_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb52-2" title="2">                                        <span class="dt">pedigree=</span>LS_Monogamy_theTree,</a>
<a class="sourceLine" id="cb52-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb52-4" title="4">                                        <span class="dt">data=</span>LS_phen_Monogamy_metaData,</a>
<a class="sourceLine" id="cb52-5" title="5">                                        <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb52-6" title="6"><span class="kw">summary</span>(LS_phen_Monogamy_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -51.18382 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.08956 0.0006984   0.3387    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.03274 0.0009689   0.0942    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.03255 0.001071  0.09425    20000

 R-structure:  ~units

      post.mean  l-95% CI u-95% CI eff.samp
units  0.003234 0.0007601 0.007181    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp pMCMC  
(Intercept)              0.51524  0.10435  0.91095    20000 0.028 *
factor(Sex)phenCV_male   0.02670 -0.02984  0.07817    20000 0.286  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC model for the genetic variance in lifespan in monogamous species (Table 1).</p>
<div class="sourceCode" id="cb54"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb54-1" title="1">LS_gen_Monogamy_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb54-2" title="2">                                       <span class="dt">pedigree=</span>LS_Monogamy_theTree,</a>
<a class="sourceLine" id="cb54-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb54-4" title="4">                                       <span class="dt">data=</span>LS_gen_Monogamy_metaData,</a>
<a class="sourceLine" id="cb54-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb54-6" title="6"><span class="kw">summary</span>(LS_gen_Monogamy_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -35.09093 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal   0.05535 0.001091   0.1965    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.01318 0.0007922  0.03887    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.01314 0.0008344  0.03826    16829

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.006671 0.001822  0.01405    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp pMCMC
(Intercept)             0.14569 -0.19152  0.47027    20000 0.261
factor(Sex)genCV_male   0.05010 -0.02676  0.12622    20000 0.174</code></pre>
<p>Then followed the models for lifespan in polygamous species. First, the MCMC for the phenotypic variance in lifespan in polygamous species (Table 1).</p>
<div class="sourceCode" id="cb56"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb56-1" title="1">LS_phen_Polygamy_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb56-2" title="2">                                        <span class="dt">pedigree=</span>LS_Polygamy_theTree,</a>
<a class="sourceLine" id="cb56-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb56-4" title="4">                                        <span class="dt">data=</span>LS_phen_Polygamy_metaData,</a>
<a class="sourceLine" id="cb56-5" title="5">                                        <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb56-6" title="6"><span class="kw">summary</span>(LS_phen_Polygamy_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -138.2843 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal    0.1165 0.001366    0.319    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index  0.009194 0.002231    0.019    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01863 0.002682  0.04381    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.003992 0.002064 0.006217    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC  
(Intercept)              0.50253  0.09672  0.93815    20000 0.0206 *
factor(Sex)phenCV_male  -0.01390 -0.04549  0.01817    20000 0.3933  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we run the MCMC model for the genetic variance in lifespan in polygamous species (Table 1).</p>
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb58-1" title="1">LS_gen_Polygamy_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb58-2" title="2">                                       <span class="dt">pedigree=</span>LS_Polygamy_theTree,</a>
<a class="sourceLine" id="cb58-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb58-4" title="4">                                       <span class="dt">data=</span>LS_gen_Polygamy_metaData,</a>
<a class="sourceLine" id="cb58-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb58-6" title="6"><span class="kw">summary</span>(LS_gen_Polygamy_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -180.7517 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal  0.009475 0.0007683  0.02627    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.001933 0.0006136 0.003849    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID  0.005425 0.001315   0.0111    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.002116 0.001202 0.003159    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean  l-95% CI  u-95% CI eff.samp  pMCMC  
(Intercept)            0.132430  0.012503  0.267101    20757 0.0401 *
factor(Sex)genCV_male  0.006762 -0.017139  0.029570    20000 0.5576  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
</div>
<div id="analyses-of-vertebrates" class="section level3">
<h3>Analyses of vertebrates</h3>
<p>In this part, we analysed the vertebrate species separately, as all monogamous species in this dataset are vertebrates (see table S5 &amp; S6).</p>
<p>We first subset our data frame to only include vertebrates (mammalia, aves or reptilia).</p>
<div class="sourceCode" id="cb60"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb60-1" title="1">Data_vertebrates &lt;-<span class="kw">subset</span>(Data, Class <span class="op">==</span><span class="st"> &quot;Mammalia&quot;</span> <span class="op">|</span><span class="st"> </span>Class <span class="op">==</span><span class="st"> &quot;Aves&quot;</span> <span class="op">|</span><span class="st"> </span>Class <span class="op">==</span><span class="st"> &quot;Reptilia&quot;</span>)</a>
<a class="sourceLine" id="cb60-2" title="2"></a>
<a class="sourceLine" id="cb60-3" title="3">stacked_gen_Data_vertebrates &lt;-<span class="st"> </span><span class="kw">gather</span>(Data_vertebrates, <span class="dt">key =</span> <span class="st">&quot;Sex&quot;</span>,<span class="dt">value =</span> <span class="st">&quot;genCV&quot;</span>, genCV_male, genCV_female)</a>
<a class="sourceLine" id="cb60-4" title="4">stacked_phen_Data_vertebrates &lt;-<span class="st"> </span><span class="kw">gather</span>(Data_vertebrates, <span class="dt">key =</span> <span class="st">&quot;Sex&quot;</span>,<span class="dt">value =</span> <span class="st">&quot;phenCV&quot;</span>, phenCV_male, phenCV_female)</a>
<a class="sourceLine" id="cb60-5" title="5"></a>
<a class="sourceLine" id="cb60-6" title="6"></a>
<a class="sourceLine" id="cb60-7" title="7">RS_gen_metaData_vertebrates&lt;-<span class="kw">subset</span>(stacked_gen_Data_vertebrates, FitnessCat <span class="op">==</span><span class="st"> &quot;RS&quot;</span>)</a>
<a class="sourceLine" id="cb60-8" title="8">LS_gen_metaData_vertebrates&lt;-<span class="kw">subset</span>(stacked_gen_Data_vertebrates, FitnessCat <span class="op">==</span><span class="st"> &quot;LS&quot;</span>)</a>
<a class="sourceLine" id="cb60-9" title="9"></a>
<a class="sourceLine" id="cb60-10" title="10">RS_phen_metaData_vertebrates&lt;-<span class="kw">subset</span>(stacked_phen_Data_vertebrates, FitnessCat <span class="op">==</span><span class="st"> &quot;RS&quot;</span>)</a>
<a class="sourceLine" id="cb60-11" title="11">LS_phen_metaData_vertebrates&lt;-<span class="kw">subset</span>(stacked_phen_Data_vertebrates, FitnessCat <span class="op">==</span><span class="st"> &quot;LS&quot;</span>)</a></code></pre></div>
<div id="mcmcs-for-reproductive-success-1" class="section level4">
<h4>MCMCs for reproductive success</h4>
<p>We pruned the phylogenetic tree to all vertebrate species with data on reproductive success.</p>
<div class="sourceCode" id="cb61"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb61-1" title="1"><span class="co">## PRUNE PHYLOGENETIC TREE TO DATA SUBSET</span></a>
<a class="sourceLine" id="cb61-2" title="2">RS_gen_metaData_vertebrates<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(RS_gen_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb61-3" title="3"><span class="kw">is.factor</span>(RS_gen_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb61-4" title="4">RS_Species_Data_vertebrates &lt;-<span class="st"> </span><span class="kw">unique</span>(RS_gen_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb61-5" title="5"><span class="kw">summary</span>(RS_Species_Data_vertebrates)</a>
<a class="sourceLine" id="cb61-6" title="6">RS_theTree_vertebrates&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(RS_Species_Data_vertebrates, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb61-7" title="7"><span class="kw">plot</span>(RS_theTree_vertebrates)</a></code></pre></div>
<div class="sourceCode" id="cb62"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb62-1" title="1">RS_phen_metaData_vertebrates<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_phen_metaData_vertebrates<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb62-2" title="2">RS_phen_metaData_vertebrates<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_phen_metaData_vertebrates<span class="op">$</span>Study_ID)</a>
<a class="sourceLine" id="cb62-3" title="3">RS_gen_metaData_vertebrates<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_gen_metaData_vertebrates<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb62-4" title="4">RS_gen_metaData_vertebrates<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(RS_gen_metaData_vertebrates<span class="op">$</span>Study_ID)</a></code></pre></div>
<p>We then first ran the MCMC models for the phenotypic variance in reproductive success in vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb63"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb63-1" title="1">RS_phen_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb63-2" title="2">                               <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb63-3" title="3">                               <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb63-4" title="4">                               <span class="dt">data=</span>RS_phen_metaData_vertebrates,</a>
<a class="sourceLine" id="cb63-5" title="5">                               <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb63-6" title="6"><span class="kw">summary</span>(RS_phen_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: 37.24732 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.09917 0.0008523   0.3995    20636

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.09962 0.001261   0.2648    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.09915 0.001095   0.2611    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.09002  0.04239   0.1505    21306

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC  
(Intercept)              0.86708  0.28132  1.45441    18529 0.0176 *
factor(Sex)phenCV_male   0.18751  0.01349  0.35834    20000 0.0345 *
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC model for the phenotypic variance in reproductive success in vertebrate species including an interaction of mating system (‘Mating_system’) and sex (Table S6).</p>
<div class="sourceCode" id="cb65"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb65-1" title="1">RS_phen_MatSyst_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb65-2" title="2">                                       <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb65-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb65-4" title="4">                                       <span class="dt">data=</span>RS_phen_metaData_vertebrates,</a>
<a class="sourceLine" id="cb65-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb65-6" title="6"><span class="kw">summary</span>(RS_phen_MatSyst_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: 11.11587 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.1391 0.0007481   0.5509    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index     0.102 0.001107    0.265    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID    0.1029 0.001023   0.2648    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.04825  0.02307  0.08173    20000

 Location effects: phenCV ~ factor(Sex) * factor(Mating_system) 

                                                     post.mean l-95% CI
(Intercept)                                            0.83082  0.12727
factor(Sex)phenCV_male                                -0.01322 -0.17314
factor(Mating_system)polygamy                          0.05426 -0.43032
factor(Sex)phenCV_male:factor(Mating_system)polygamy   0.57623  0.31328
                                                     u-95% CI eff.samp  pMCMC
(Intercept)                                           1.49595    20000 0.0366
factor(Sex)phenCV_male                                0.14936    20000 0.8575
factor(Mating_system)polygamy                         0.54381    20000 0.8275
factor(Sex)phenCV_male:factor(Mating_system)polygamy  0.85478    20000 0.0005
                                                        
(Intercept)                                          *  
factor(Sex)phenCV_male                                  
factor(Mating_system)polygamy                           
factor(Sex)phenCV_male:factor(Mating_system)polygamy ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Then we ran the MCMC models for the genetic variance in reproductive success in vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb67"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb67-1" title="1">RS_gen_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb67-2" title="2">                              <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb67-3" title="3">                              <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb67-4" title="4">                              <span class="dt">data=</span>RS_gen_metaData_vertebrates,</a>
<a class="sourceLine" id="cb67-5" title="5">                              <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb67-6" title="6"><span class="kw">summary</span>(RS_gen_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -33.89924 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.03326 0.0008146   0.1174    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.01177 0.001017  0.02845    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01181 0.001018  0.02876    20453

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.02015   0.0101  0.03242    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean  l-95% CI  u-95% CI eff.samp  pMCMC  
(Intercept)            0.178203 -0.133438  0.511197    20749 0.1796  
factor(Sex)genCV_male  0.084344  0.003013  0.167764    20000 0.0458 *
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we run the MCMC model for the genetic variance in reproductive success in vertebrate species including an interaction of mating system (‘Mating_system’) and sex (Table S6).</p>
<div class="sourceCode" id="cb69"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb69-1" title="1">RS_gen_MatSyst_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb69-2" title="2">                                      <span class="dt">pedigree=</span>RS_theTree,</a>
<a class="sourceLine" id="cb69-3" title="3">                                      <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb69-4" title="4">                                      <span class="dt">data=</span>RS_gen_metaData_vertebrates,</a>
<a class="sourceLine" id="cb69-5" title="5">                                      <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb69-6" title="6"><span class="kw">summary</span>(RS_gen_MatSyst_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -58.96858 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.03997 0.0006986   0.1471    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.01375 0.001176  0.03242    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01386 0.001234  0.03276    19031

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01093 0.005139  0.01794    19572

 Location effects: genCV ~ factor(Sex) * factor(Mating_system) 

                                                    post.mean l-95% CI u-95% CI
(Intercept)                                           0.21437 -0.14739  0.57569
factor(Sex)genCV_male                                -0.01488 -0.09030  0.06004
factor(Mating_system)polygamy                        -0.10892 -0.30136  0.08809
factor(Sex)genCV_male:factor(Mating_system)polygamy   0.28480  0.15776  0.41318
                                                    eff.samp  pMCMC    
(Intercept)                                            20000 0.1553    
factor(Sex)genCV_male                                  19179 0.6859    
factor(Mating_system)polygamy                          20000 0.2653    
factor(Sex)genCV_male:factor(Mating_system)polygamy    20000 0.0002 ***
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
<div id="mcmcs-for-lifespan-1" class="section level4">
<h4>MCMCs for lifespan</h4>
<p>Now followed the vertebrate specific MCMCs for lifespan. We pruned the phylogenetic tree to all vertebrate species with data on lifespan.</p>
<div class="sourceCode" id="cb71"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb71-1" title="1"><span class="co">## PRUNE PHYLOGENETIC TREE TO DATA SUBSET</span></a>
<a class="sourceLine" id="cb71-2" title="2">LS_gen_metaData_vertebrates<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(LS_gen_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb71-3" title="3"><span class="kw">is.factor</span>(LS_gen_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb71-4" title="4">LS_Species_Data_vertebrates &lt;-<span class="st"> </span><span class="kw">unique</span>(LS_gen_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb71-5" title="5"><span class="kw">summary</span>(LS_Species_Data_vertebrates)</a>
<a class="sourceLine" id="cb71-6" title="6">LS_theTree_vertebrates&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(LS_Species_Data_vertebrates, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb71-7" title="7"><span class="kw">plot</span>(LS_theTree_vertebrates)</a></code></pre></div>
<div class="sourceCode" id="cb72"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb72-1" title="1">LS_phen_metaData_vertebrates<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(LS_phen_metaData_vertebrates<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb72-2" title="2">LS_phen_metaData_vertebrates<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(LS_phen_metaData_vertebrates<span class="op">$</span>Study_ID)</a>
<a class="sourceLine" id="cb72-3" title="3">LS_gen_metaData_vertebrates<span class="op">$</span>Index &lt;-<span class="st"> </span><span class="kw">as.factor</span>(LS_gen_metaData_vertebrates<span class="op">$</span>Index)</a>
<a class="sourceLine" id="cb72-4" title="4">LS_gen_metaData_vertebrates<span class="op">$</span>Study_ID &lt;-<span class="st"> </span><span class="kw">as.factor</span>(LS_gen_metaData_vertebrates<span class="op">$</span>Study_ID)</a></code></pre></div>
<p>We then ran the MCMC models, first for the phenotypic variance in lifespan in vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb73"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb73-1" title="1">LS_phen_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb73-2" title="2">                                           <span class="dt">pedigree=</span>LS_theTree,</a>
<a class="sourceLine" id="cb73-3" title="3">                                           <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb73-4" title="4">                                           <span class="dt">data=</span>LS_phen_metaData_vertebrates,</a>
<a class="sourceLine" id="cb73-5" title="5">                                           <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb73-6" title="6"><span class="kw">summary</span>(LS_phen_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -65.06646 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.1863 0.0008099   0.8199    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.04021 0.0009848   0.1085    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.04051 0.0008569   0.1119    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.003681 0.001224 0.007095    21065

 Location effects: phenCV ~ factor(Sex) 

                       post.mean  l-95% CI  u-95% CI eff.samp  pMCMC  
(Intercept)             0.549536 -0.201479  1.336565    20504 0.0953 .
factor(Sex)phenCV_male -0.003234 -0.049740  0.043387    20000 0.8842  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC model for the phenotypic variance in lifespan in vertebrate species including an interaction of mating system (‘Mating_system’) and sex (Table S6).</p>
<div class="sourceCode" id="cb75"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb75-1" title="1">LS_phen_MatSyst_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb75-2" title="2">                                                   <span class="dt">pedigree=</span>LS_theTree,</a>
<a class="sourceLine" id="cb75-3" title="3">                                                   <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb75-4" title="4">                                                   <span class="dt">data=</span>LS_phen_metaData_vertebrates,</a>
<a class="sourceLine" id="cb75-5" title="5">                                                   <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb75-6" title="6"><span class="kw">summary</span>(LS_phen_MatSyst_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -74.97083 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.3496 0.0007483    1.387    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.03932 0.000838   0.1129    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID    0.0386 0.0009624   0.1104    20000

 R-structure:  ~units

      post.mean  l-95% CI u-95% CI eff.samp
units   0.00262 0.0008446 0.005217    20000

 Location effects: phenCV ~ factor(Sex) * factor(Mating_system) 

                                                     post.mean l-95% CI
(Intercept)                                            0.46957 -0.66281
factor(Sex)phenCV_male                                 0.02686 -0.01946
factor(Mating_system)polygamy                          0.23259 -0.28535
factor(Sex)phenCV_male:factor(Mating_system)polygamy  -0.09772 -0.18490
                                                     u-95% CI eff.samp  pMCMC  
(Intercept)                                           1.47749    20425 0.2232  
factor(Sex)phenCV_male                                0.07562    20000 0.2382  
factor(Mating_system)polygamy                         0.74345    20000 0.3333  
factor(Sex)phenCV_male:factor(Mating_system)polygamy -0.01288    20000 0.0307 *
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Then we ran the MCMC models for the genetic variance in lifespan in vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb77"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb77-1" title="1">LS_gen_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb77-2" title="2">                                          <span class="dt">pedigree=</span>LS_theTree,</a>
<a class="sourceLine" id="cb77-3" title="3">                                          <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb77-4" title="4">                                          <span class="dt">data=</span>LS_gen_metaData_vertebrates,</a>
<a class="sourceLine" id="cb77-5" title="5">                                          <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb77-6" title="6"><span class="kw">summary</span>(LS_gen_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -48.87279 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.05246 0.0008486    0.183    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.009012 0.0009802  0.02369    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID  0.008917 0.0008867  0.02314    19646

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.006555 0.002279  0.01211    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp pMCMC
(Intercept)             0.12917 -0.28251  0.54413    20451 0.376
factor(Sex)genCV_male   0.02424 -0.03988  0.08738    20000 0.421</code></pre>
<p>Next, we ran the MCMC model for the genetic variance in lifespan in vertebrate species including an interaction of mating system (‘Mating_system’) and sex (Table S6).</p>
<div class="sourceCode" id="cb79"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb79-1" title="1">LS_gen_MatSyst_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(Mating_system),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb79-2" title="2">                                                  <span class="dt">pedigree=</span>LS_theTree,</a>
<a class="sourceLine" id="cb79-3" title="3">                                                  <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb79-4" title="4">                                                  <span class="dt">data=</span>LS_gen_metaData_vertebrates,</a>
<a class="sourceLine" id="cb79-5" title="5">                                                  <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb79-6" title="6"><span class="kw">summary</span>(LS_gen_MatSyst_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -49.03527 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.06092 0.0007855   0.2133    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.009688 0.0008992  0.02576    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID  0.009616 0.0008963  0.02511    19804

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.006402 0.002061  0.01211    21243

 Location effects: genCV ~ factor(Sex) * factor(Mating_system) 

                                                    post.mean l-95% CI u-95% CI
(Intercept)                                           0.13206 -0.31702  0.57926
factor(Sex)genCV_male                                 0.04948 -0.02302  0.12679
factor(Mating_system)polygamy                        -0.02164 -0.26209  0.22106
factor(Sex)genCV_male:factor(Mating_system)polygamy  -0.08192 -0.21146  0.05496
                                                    eff.samp pMCMC
(Intercept)                                            20000 0.388
factor(Sex)genCV_male                                  18648 0.181
factor(Mating_system)polygamy                          20463 0.830
factor(Sex)genCV_male:factor(Mating_system)polygamy    20000 0.210</code></pre>
</div>
</div>
<div id="analyses-of-vertebrates-by-mating-system" class="section level3">
<h3>Analyses of vertebrates by mating system</h3>
<p>Finally, we analysed the vertebrate species split by mating system.</p>
<p>We first subset our vertebrates data frame into socially monogamous and polygamous species.</p>
<div class="sourceCode" id="cb81"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb81-1" title="1"><span class="co">## additional subsetting</span></a>
<a class="sourceLine" id="cb81-2" title="2">RS_gen_Monogamy_metaData_vertebrates&lt;-<span class="kw">subset</span>(RS_gen_metaData_vertebrates, Mating_system <span class="op">==</span><span class="st"> &quot;monogamy&quot;</span>)</a>
<a class="sourceLine" id="cb81-3" title="3">RS_gen_Polygamy_metaData_vertebrates&lt;-<span class="kw">subset</span>(RS_gen_metaData_vertebrates, Mating_system <span class="op">==</span><span class="st"> &quot;polygamy&quot;</span>)</a>
<a class="sourceLine" id="cb81-4" title="4"></a>
<a class="sourceLine" id="cb81-5" title="5">RS_phen_Monogamy_metaData_vertebrates&lt;-<span class="kw">subset</span>(RS_phen_metaData_vertebrates, Mating_system <span class="op">==</span><span class="st"> &quot;monogamy&quot;</span>)</a>
<a class="sourceLine" id="cb81-6" title="6">RS_phen_Polygamy_metaData_vertebrates&lt;-<span class="kw">subset</span>(RS_phen_metaData_vertebrates, Mating_system <span class="op">==</span><span class="st"> &quot;polygamy&quot;</span>)</a>
<a class="sourceLine" id="cb81-7" title="7"></a>
<a class="sourceLine" id="cb81-8" title="8">LS_gen_Monogamy_metaData_vertebrates&lt;-<span class="kw">subset</span>(LS_gen_metaData_vertebrates, Mating_system <span class="op">==</span><span class="st"> &quot;monogamy&quot;</span>)</a>
<a class="sourceLine" id="cb81-9" title="9">LS_gen_Polygamy_metaData_vertebrates&lt;-<span class="kw">subset</span>(LS_gen_metaData_vertebrates, Mating_system <span class="op">==</span><span class="st"> &quot;polygamy&quot;</span>)</a>
<a class="sourceLine" id="cb81-10" title="10"></a>
<a class="sourceLine" id="cb81-11" title="11">LS_phen_Monogamy_metaData_vertebrates&lt;-<span class="kw">subset</span>(LS_phen_metaData_vertebrates, Mating_system <span class="op">==</span><span class="st"> &quot;monogamy&quot;</span>)</a>
<a class="sourceLine" id="cb81-12" title="12">LS_phen_Polygamy_metaData_vertebrates&lt;-<span class="kw">subset</span>(LS_phen_metaData_vertebrates, Mating_system <span class="op">==</span><span class="st"> &quot;polygamy&quot;</span>)</a></code></pre></div>
<p>We prune the phylogenetic trees to all vertebrate species with data on reproductive success or lifespan for monogamous and polygamous species separately.</p>
<div class="sourceCode" id="cb82"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb82-1" title="1"><span class="co">## PRUNE PHYLOGENETIC TREES</span></a>
<a class="sourceLine" id="cb82-2" title="2">RS_gen_Monogamy_metaData_vertebrates<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(RS_gen_Monogamy_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb82-3" title="3">RS_Monogamy_Species_Data_vertebrates &lt;-<span class="st"> </span><span class="kw">unique</span>(RS_gen_Monogamy_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb82-4" title="4"><span class="kw">summary</span>(RS_Monogamy_Species_Data_vertebrates)</a>
<a class="sourceLine" id="cb82-5" title="5">RS_Monogamy_theTree_vertebrates&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(RS_Monogamy_Species_Data_vertebrates, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb82-6" title="6"><span class="kw">plot</span>(RS_Monogamy_theTree_vertebrates)</a></code></pre></div>
<div class="sourceCode" id="cb83"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb83-1" title="1">RS_gen_Polygamy_metaData_vertebrates<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(RS_gen_Polygamy_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb83-2" title="2">RS_Polygamy_Species_Data_vertebrates &lt;-<span class="st"> </span><span class="kw">unique</span>(RS_gen_Polygamy_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb83-3" title="3"><span class="kw">summary</span>(RS_Polygamy_Species_Data_vertebrates)</a>
<a class="sourceLine" id="cb83-4" title="4">RS_Polygamy_theTree_vertebrates&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(RS_Polygamy_Species_Data_vertebrates, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb83-5" title="5"><span class="kw">plot</span>(RS_Polygamy_theTree_vertebrates)</a></code></pre></div>
<div class="sourceCode" id="cb84"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb84-1" title="1">LS_gen_Monogamy_metaData_vertebrates<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(LS_gen_Monogamy_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb84-2" title="2">LS_Monogamy_Species_Data_vertebrates &lt;-<span class="st"> </span><span class="kw">unique</span>(LS_gen_Monogamy_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb84-3" title="3"><span class="kw">summary</span>(LS_Monogamy_Species_Data_vertebrates)</a>
<a class="sourceLine" id="cb84-4" title="4">LS_Monogamy_theTree_vertebrates&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(LS_Monogamy_Species_Data_vertebrates, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb84-5" title="5"><span class="kw">plot</span>(LS_Monogamy_theTree_vertebrates)</a></code></pre></div>
<div class="sourceCode" id="cb85"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb85-1" title="1">LS_gen_Polygamy_metaData_vertebrates<span class="op">$</span>animal &lt;-<span class="st"> </span><span class="kw">factor</span>(LS_gen_Polygamy_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb85-2" title="2">LS_Polygamy_Species_Data_vertebrates &lt;-<span class="st"> </span><span class="kw">unique</span>(LS_gen_Polygamy_metaData_vertebrates<span class="op">$</span>animal)</a>
<a class="sourceLine" id="cb85-3" title="3"><span class="kw">summary</span>(LS_Polygamy_Species_Data_vertebrates)</a>
<a class="sourceLine" id="cb85-4" title="4">LS_Polygamy_theTree_vertebrates&lt;-<span class="kw">drop.tip</span>(theTree, theTree<span class="op">$</span>tip.label[<span class="op">-</span><span class="kw">na.omit</span>(<span class="kw">match</span>(LS_Polygamy_Species_Data_vertebrates, theTree<span class="op">$</span>tip.label))])</a>
<a class="sourceLine" id="cb85-5" title="5"><span class="kw">plot</span>(LS_Polygamy_theTree_vertebrates)</a></code></pre></div>
<div id="mcmcs-for-reproductive-success-2" class="section level4">
<h4>MCMCs for reproductive success</h4>
<p>We then first ran the MCMC models for the phenotypic variance in reproductive success in socially monogamous vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb86"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb86-1" title="1">RS_phen_Monogamy_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb86-2" title="2">                                        <span class="dt">pedigree=</span>RS_Monogamy_theTree,</a>
<a class="sourceLine" id="cb86-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb86-4" title="4">                                        <span class="dt">data=</span>RS_phen_Monogamy_metaData_vertebrates,</a>
<a class="sourceLine" id="cb86-5" title="5">                                        <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb86-6" title="6"><span class="kw">summary</span>(RS_phen_Monogamy_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -65.82395 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.5273 0.0006507    2.504    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.09523 0.0006839   0.2618    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID    0.0926 0.0007946   0.2521    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.004651  0.00176 0.008672    21379

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC  
(Intercept)              0.84744 -0.17942  1.77799    20199 0.0707 .
factor(Sex)phenCV_male  -0.01260 -0.06223  0.03545    20000 0.5853  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC model for the genetic variance in reproductive success in socially monogamous vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb88"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb88-1" title="1">RS_gen_Monogamy_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb88-2" title="2">                                       <span class="dt">pedigree=</span>RS_Monogamy_theTree,</a>
<a class="sourceLine" id="cb88-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb88-4" title="4">                                       <span class="dt">data=</span>RS_gen_Monogamy_metaData_vertebrates,</a>
<a class="sourceLine" id="cb88-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb88-6" title="6"><span class="kw">summary</span>(RS_gen_Monogamy_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -51.49018 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.06302 0.0008169   0.2485    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.01259 0.0008919  0.03188    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01263 0.001057   0.0324    20551

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.00753  0.00298  0.01386    19966

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp pMCMC
(Intercept)             0.21227 -0.14167  0.54623    20000 0.130
factor(Sex)genCV_male  -0.01510 -0.07858  0.04649    18383 0.625</code></pre>
<p>Then we ran the MCMC models for the phenotypic variance in reproductive success in polygamous vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb90"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb90-1" title="1">RS_phen_Polygamy_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb90-2" title="2">                                        <span class="dt">pedigree=</span>RS_Polygamy_theTree,</a>
<a class="sourceLine" id="cb90-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb90-4" title="4">                                        <span class="dt">data=</span>RS_phen_Polygamy_metaData_vertebrates,</a>
<a class="sourceLine" id="cb90-5" title="5">                                        <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb90-6" title="6"><span class="kw">summary</span>(RS_phen_Polygamy_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: 21.56509 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.3819 0.0008716    1.605    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index    0.1693 0.0007676   0.5853    19001

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID    0.1682 0.0006695    0.598    19839

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units    0.1734  0.03728   0.3935    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC  
(Intercept)               0.8751  -0.1849   1.9941    20000 0.0800 .
factor(Sex)phenCV_male    0.5599   0.1538   0.9827    20000 0.0137 *
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC model for the genetic variance in reproductive success in polygamous vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb92"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb92-1" title="1">RS_gen_Polygamy_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb92-2" title="2">                                       <span class="dt">pedigree=</span>RS_Polygamy_theTree,</a>
<a class="sourceLine" id="cb92-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb92-4" title="4">                                       <span class="dt">data=</span>RS_gen_Polygamy_metaData_vertebrates,</a>
<a class="sourceLine" id="cb92-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb92-6" title="6"><span class="kw">summary</span>(RS_gen_Polygamy_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -10.15737 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal    0.1041 0.0008755    0.386    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.03625 0.0006739   0.1148    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.03579 0.0009617   0.1135    20915

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.02383 0.005593   0.0537    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp  pMCMC   
(Intercept)              0.1021  -0.5018   0.6074    20000 0.5725   
factor(Sex)genCV_male    0.2703   0.1163   0.4290    20000 0.0032 **
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
<div id="mcmcs-for-lifespan-2" class="section level4">
<h4>MCMCs for lifespan</h4>
<p>Now followed the mating system specific MCMCs for lifespan in vertebrates. First, the MCMC for the phenotypic variance in lifespan in socially monogamous vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb94"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb94-1" title="1">LS_phen_Monogamy_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb94-2" title="2">                                        <span class="dt">pedigree=</span>LS_Monogamy_theTree,</a>
<a class="sourceLine" id="cb94-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb94-4" title="4">                                        <span class="dt">data=</span>LS_phen_Monogamy_metaData_vertebrates,</a>
<a class="sourceLine" id="cb94-5" title="5">                                        <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb94-6" title="6"><span class="kw">summary</span>(LS_phen_Monogamy_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -51.18958 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.09316 0.0006337    0.338    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index    0.0328 0.0006974  0.09674    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID    0.0327 0.0009569  0.09598    20000

 R-structure:  ~units

      post.mean  l-95% CI u-95% CI eff.samp
units  0.003209 0.0007803 0.007097    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp  pMCMC  
(Intercept)              0.51014  0.07754  0.91852    20000 0.0323 *
factor(Sex)phenCV_male   0.02688 -0.02605  0.07985    20000 0.2758  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC model for the genetic variance in lifespan in socially monogamous vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb96"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb96-1" title="1">LS_gen_Monogamy_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb96-2" title="2">                                       <span class="dt">pedigree=</span>LS_Monogamy_theTree,</a>
<a class="sourceLine" id="cb96-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb96-4" title="4">                                       <span class="dt">data=</span>LS_gen_Monogamy_metaData_vertebrates,</a>
<a class="sourceLine" id="cb96-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb96-6" title="6"><span class="kw">summary</span>(LS_gen_Monogamy_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -35.08729 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.05737 0.0007049   0.1971    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.01321 0.000762  0.03857    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.01307 0.0008188  0.03725    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.006645 0.001789  0.01379    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp pMCMC
(Intercept)             0.14564 -0.18708  0.46842    20000 0.265
factor(Sex)genCV_male   0.04893 -0.02428  0.13013    19025 0.187</code></pre>
<p>Then we ran the MCMC models for the phenotypic variance in lifespan in polygamous vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb98"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb98-1" title="1">LS_phen_Polygamy_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb98-2" title="2">                                        <span class="dt">pedigree=</span>LS_Polygamy_theTree,</a>
<a class="sourceLine" id="cb98-3" title="3">                                        <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb98-4" title="4">                                        <span class="dt">data=</span>LS_phen_Polygamy_metaData_vertebrates,</a>
<a class="sourceLine" id="cb98-5" title="5">                                        <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb98-6" title="6"><span class="kw">summary</span>(LS_phen_Polygamy_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -18.7178 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal     5.929 0.0008171    12.06    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index    0.2756 0.0007145   0.8335    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID    0.2835 0.000883   0.8488    20000

 R-structure:  ~units

      post.mean  l-95% CI u-95% CI eff.samp
units   0.01186 0.0007259  0.03687    20000

 Location effects: phenCV ~ factor(Sex) 

                       post.mean l-95% CI u-95% CI eff.samp pMCMC
(Intercept)              0.71490 -2.08563  3.65561    20000 0.305
factor(Sex)phenCV_male  -0.07031 -0.21813  0.07990    20000 0.246</code></pre>
<p>Next, we ran the MCMC model for the genetic variance in lifespan in polygamous vertebrate species (Table S5).</p>
<div class="sourceCode" id="cb100"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb100-1" title="1">LS_gen_Polygamy_MCMC_model_vertebrates &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb100-2" title="2">                                       <span class="dt">pedigree=</span>LS_Polygamy_theTree,</a>
<a class="sourceLine" id="cb100-3" title="3">                                       <span class="dt">prior=</span>pr,</a>
<a class="sourceLine" id="cb100-4" title="4">                                       <span class="dt">data=</span>LS_gen_Polygamy_metaData_vertebrates,</a>
<a class="sourceLine" id="cb100-5" title="5">                                       <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb100-6" title="6"><span class="kw">summary</span>(LS_gen_Polygamy_MCMC_model_vertebrates)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -11.98633 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal     3.459 0.0005238    2.576    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index   0.09021 0.0006282   0.2718    20000

               ~Study_ID

         post.mean  l-95% CI u-95% CI eff.samp
Study_ID   0.08062 0.0006904   0.2611    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01735 0.001496  0.05143    20000

 Location effects: genCV ~ factor(Sex) 

                      post.mean l-95% CI u-95% CI eff.samp pMCMC
(Intercept)             0.13778 -1.12651  1.45733    20000 0.649
factor(Sex)genCV_male  -0.03284 -0.21857  0.15540    19017 0.643</code></pre>
</div>
</div>
<div id="methodological-correlates" class="section level3">
<h3>Methodological correlates</h3>
<p>In this final part of the analysis, we explored if the methodological moderators we collected interacted with the effect of sex on phenotypic and genetic variances. This was done to exclude that the methodological diversity in the data confounded the results. We explored effects of the study type (lab versus field studies) (see table S7), the estimate of the genetic variance (additive versus total VG) (see table S8) and the reproductive success estimate (annual versus lifetime reproductive success) (see table S9).</p>
<div id="study-type" class="section level4">
<h4>Study type</h4>
<p>First, we explored the effect of study type (lab versus field studies). We began with the phenotypic variance in reproductive success (Table S7).</p>
<div class="sourceCode" id="cb102"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb102-1" title="1">RS_phen_StudyType_MCMC_model  &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(StudyType),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb102-2" title="2">                                          <span class="dt">pedigree=</span>RS_theTree,<span class="dt">data=</span>RS_phen_metaData,</a>
<a class="sourceLine" id="cb102-3" title="3">                                          <span class="dt">prior=</span>pr, <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb102-4" title="4"><span class="kw">summary</span>(RS_phen_StudyType_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: 50.73778 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal   0.02394 0.000909   0.0798    19202

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.05878 0.003917   0.1182    20753

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.06025 0.001297   0.1425    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.05988  0.03933  0.08262    20000

 Location effects: phenCV ~ factor(Sex) * factor(StudyType) 

                                                   post.mean l-95% CI u-95% CI
(Intercept)                                          0.86636  0.54772  1.17815
factor(Sex)phenCV_male                               0.18773  0.04862  0.33064
factor(StudyType)Laboratory                         -0.30426 -0.70391  0.12134
factor(Sex)phenCV_male:factor(StudyType)Laboratory   0.07385 -0.10323  0.25220
                                                   eff.samp  pMCMC    
(Intercept)                                           20000 0.0002 ***
factor(Sex)phenCV_male                                20000 0.0091 ** 
factor(StudyType)Laboratory                           21409 0.1259    
factor(Sex)phenCV_male:factor(StudyType)Laboratory    20139 0.4069    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC exploring the effect of study type on the genetic variance in reproductive success (Table S7).</p>
<div class="sourceCode" id="cb104"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb104-1" title="1">RS_gen_StudyType_MCMC_model  &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(StudyType),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb104-2" title="2">                                         <span class="dt">pedigree=</span>RS_theTree,<span class="dt">data=</span>RS_gen_metaData,</a>
<a class="sourceLine" id="cb104-3" title="3">                                         <span class="dt">prior=</span>pr, <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb104-4" title="4"><span class="kw">summary</span>(RS_gen_StudyType_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -128.5008 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal   0.01913 0.001083  0.05267    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index  0.005768 0.001092  0.01197    19575

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01382 0.003778  0.02563    18904

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01492  0.01011  0.02001    20000

 Location effects: genCV ~ factor(Sex) * factor(StudyType) 

                                                  post.mean  l-95% CI  u-95% CI
(Intercept)                                        0.177840 -0.080698  0.426172
factor(Sex)genCV_male                              0.084467  0.010894  0.153118
factor(StudyType)Laboratory                        0.133421 -0.178377  0.464840
factor(Sex)genCV_male:factor(StudyType)Laboratory  0.003481 -0.085962  0.093480
                                                  eff.samp  pMCMC  
(Intercept)                                          20000 0.1313  
factor(Sex)genCV_male                                20000 0.0192 *
factor(StudyType)Laboratory                          20000 0.3494  
factor(Sex)genCV_male:factor(StudyType)Laboratory    20000 0.9339  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Then we ran the model for the phenotypic variance in lifespan (Table S7).</p>
<div class="sourceCode" id="cb106"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb106-1" title="1">LS_phen_StudyType_MCMC_model  &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(StudyType),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb106-2" title="2">                                          <span class="dt">pedigree=</span>LS_theTree,<span class="dt">data=</span>LS_phen_metaData,</a>
<a class="sourceLine" id="cb106-3" title="3">                                          <span class="dt">prior=</span>pr, <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb106-4" title="4"><span class="kw">summary</span>(LS_phen_StudyType_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -186.5848 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.03937 0.0008584   0.1392    20037

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.01125 0.002594  0.02356    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.02789 0.004757  0.05386    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.003505 0.002009  0.00519    20000

 Location effects: phenCV ~ factor(Sex) * factor(StudyType) 

                                                   post.mean  l-95% CI
(Intercept)                                         0.547094  0.175436
factor(Sex)phenCV_male                             -0.003396 -0.047082
factor(StudyType)Laboratory                        -0.222072 -0.706827
factor(Sex)phenCV_male:factor(StudyType)Laboratory -0.002088 -0.060325
                                                    u-95% CI eff.samp  pMCMC  
(Intercept)                                         0.915865    19723 0.0161 *
factor(Sex)phenCV_male                              0.044355    20449 0.8802  
factor(StudyType)Laboratory                         0.311116    20532 0.2685  
factor(Sex)phenCV_male:factor(StudyType)Laboratory  0.052921    22732 0.9370  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Finally, we explored the effect of study type on the genetic variance in lifespan (Table S7).</p>
<div class="sourceCode" id="cb108"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb108-1" title="1">LS_gen_StudyType_MCMC_model  &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex) <span class="op">*</span><span class="st"> </span><span class="kw">factor</span>(StudyType),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb108-2" title="2">                                         <span class="dt">pedigree=</span>LS_theTree,<span class="dt">data=</span>LS_gen_metaData,</a>
<a class="sourceLine" id="cb108-3" title="3">                                         <span class="dt">prior=</span>pr, <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb108-4" title="4"><span class="kw">summary</span>(LS_gen_StudyType_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -212.1104 

 G-structure:  ~animal

       post.mean l-95% CI u-95% CI eff.samp
animal   0.01743 0.001033  0.04831    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.002139 0.0006318 0.004284    21003

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID  0.006147 0.001715  0.01191    19583

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.002703 0.001667 0.003941    20000

 Location effects: genCV ~ factor(Sex) * factor(StudyType) 

                                                  post.mean l-95% CI u-95% CI
(Intercept)                                         0.12815 -0.11335  0.37351
factor(Sex)genCV_male                               0.02440 -0.01496  0.06560
factor(StudyType)Laboratory                         0.01941 -0.29047  0.36505
factor(Sex)genCV_male:factor(StudyType)Laboratory  -0.01150 -0.06159  0.03463
                                                  eff.samp pMCMC
(Intercept)                                          19568 0.224
factor(Sex)genCV_male                                19762 0.221
factor(StudyType)Laboratory                          20000 0.884
factor(Sex)genCV_male:factor(StudyType)Laboratory    20000 0.641</code></pre>
</div>
<div id="fitness-estimate" class="section level4">
<h4>Fitness estimate</h4>
<p>Secondly, we explored the effect of the type of estimate of the genetic variance (additive versus total VG). We began with the phenotypic variance in reproductive success (Table S8).</p>
<div class="sourceCode" id="cb110"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb110-1" title="1">RS_phen_FitnessEstimate_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(phenCV<span class="op">~</span><span class="kw">factor</span>(Sex)<span class="op">*</span><span class="kw">factor</span>(FitnessEstimate),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb110-2" title="2">                                               <span class="dt">pedigree=</span>RS_theTree,<span class="dt">data=</span>RS_phen_metaData,</a>
<a class="sourceLine" id="cb110-3" title="3">                                               <span class="dt">prior=</span>pr, <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb110-4" title="4"><span class="kw">summary</span>(RS_phen_FitnessEstimate_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: 45.67907 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.03042 0.0008524  0.09391    19728

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.05654 0.006369   0.1127    20571

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.05352 0.001505    0.129    20477

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.05757  0.03826  0.07968    20000

 Location effects: phenCV ~ factor(Sex) * factor(FitnessEstimate) 

                                                 post.mean l-95% CI u-95% CI
(Intercept)                                        0.80976  0.55724  1.07860
factor(Sex)phenCV_male                             0.31143  0.19164  0.43492
factor(FitnessEstimate)RS                         -0.19289 -0.43894  0.06645
factor(Sex)phenCV_male:factor(FitnessEstimate)RS  -0.15042 -0.31715  0.01819
                                                 eff.samp  pMCMC    
(Intercept)                                         20000 &lt;5e-05 ***
factor(Sex)phenCV_male                              20732 &lt;5e-05 ***
factor(FitnessEstimate)RS                           20000 0.1283    
factor(Sex)phenCV_male:factor(FitnessEstimate)RS    20907 0.0769 .  
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC exploring the effect of the fitness estimate on the genetic variance in reproductive success (Table S8).</p>
<div class="sourceCode" id="cb112"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb112-1" title="1">RS_gen_FitnessEstimate_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex)<span class="op">*</span><span class="kw">factor</span>(FitnessEstimate),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb112-2" title="2">                                              <span class="dt">pedigree=</span>RS_theTree,<span class="dt">data=</span>RS_gen_metaData,</a>
<a class="sourceLine" id="cb112-3" title="3">                                              <span class="dt">prior=</span>pr, <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb112-4" title="4"><span class="kw">summary</span>(RS_gen_FitnessEstimate_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -132.1957 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01745 0.0009907  0.04789    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index   0.00591 0.001183  0.01215    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01481 0.004042  0.02683    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01433 0.009856  0.01934    20000

 Location effects: genCV ~ factor(Sex) * factor(FitnessEstimate) 

                                                post.mean l-95% CI u-95% CI
(Intercept)                                       0.23599  0.07649  0.41143
factor(Sex)genCV_male                             0.12224  0.06256  0.18442
factor(FitnessEstimate)RS                         0.04032 -0.08714  0.15852
factor(Sex)genCV_male:factor(FitnessEstimate)RS  -0.06945 -0.15458  0.01544
                                                eff.samp  pMCMC    
(Intercept)                                        21077 0.0085 ** 
factor(Sex)genCV_male                              20502 0.0003 ***
factor(FitnessEstimate)RS                          20000 0.5190    
factor(Sex)genCV_male:factor(FitnessEstimate)RS    19841 0.1075    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
</div>
<div id="reproductive-success-estimate" class="section level4">
<h4>Reproductive success estimate</h4>
<p>Finally, we explored the effect of the reproductive success estimate (annual versus lifetime reproductive success). We began with the genetic variance in reproductive success (Table S9).</p>
<div class="sourceCode" id="cb114"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb114-1" title="1">RS_gen_GenVar_estimate_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex)<span class="op">*</span><span class="kw">factor</span>(GenVar_estimate),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb114-2" title="2">                                              <span class="dt">pedigree=</span>RS_theTree,<span class="dt">data=</span>RS_gen_metaData,</a>
<a class="sourceLine" id="cb114-3" title="3">                                              <span class="dt">prior=</span>pr, <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb114-4" title="4"><span class="kw">summary</span>(RS_gen_GenVar_estimate_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -132.0157 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01949 0.0009287  0.05511    20000

               ~Index

      post.mean l-95% CI u-95% CI eff.samp
Index  0.005745 0.001284    0.012    19055

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID   0.01424 0.003951   0.0262    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units   0.01441 0.009746  0.01931    20000

 Location effects: genCV ~ factor(Sex) * factor(GenVar_estimate) 

                                                          post.mean l-95% CI
(Intercept)                                                 0.28240  0.10999
factor(Sex)genCV_male                                       0.11817  0.05932
factor(GenVar_estimate)narrow-sense                        -0.03087 -0.17521
factor(Sex)genCV_male:factor(GenVar_estimate)narrow-sense  -0.06313 -0.15037
                                                          u-95% CI eff.samp
(Intercept)                                                0.47747    20000
factor(Sex)genCV_male                                      0.17861    20000
factor(GenVar_estimate)narrow-sense                        0.11547    20000
factor(Sex)genCV_male:factor(GenVar_estimate)narrow-sense  0.01933    20243
                                                           pMCMC    
(Intercept)                                               0.0046 ** 
factor(Sex)genCV_male                                     0.0003 ***
factor(GenVar_estimate)narrow-sense                       0.6737    
factor(Sex)genCV_male:factor(GenVar_estimate)narrow-sense 0.1397    
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1</code></pre>
<p>Next, we ran the MCMC exploring the effect of the reproductive success estimate on the genetic variance in lifespan (Table S9).</p>
<div class="sourceCode" id="cb116"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb116-1" title="1">LS_gen_GenVar_estimate_MCMC_model &lt;-<span class="st"> </span><span class="kw">MCMCglmm</span>(genCV<span class="op">~</span><span class="kw">factor</span>(Sex)<span class="op">*</span><span class="kw">factor</span>(GenVar_estimate),<span class="dt">random=</span><span class="op">~</span>animal <span class="op">+</span><span class="st"> </span>Index <span class="op">+</span><span class="st"> </span>Study_ID,</a>
<a class="sourceLine" id="cb116-2" title="2">                                              <span class="dt">pedigree=</span>LS_theTree,<span class="dt">data=</span>LS_gen_metaData,</a>
<a class="sourceLine" id="cb116-3" title="3">                                              <span class="dt">prior=</span>pr, <span class="dt">pr =</span> <span class="ot">TRUE</span>, <span class="dt">burnin =</span> BURNIN, <span class="dt">nitt=</span>NITT, <span class="dt">thin=</span>THIN)</a>
<a class="sourceLine" id="cb116-4" title="4"><span class="kw">summary</span>(LS_gen_GenVar_estimate_MCMC_model)</a></code></pre></div>
<pre><code>
 Iterations = 1000001:10999501
 Thinning interval  = 500
 Sample size  = 20000 

 DIC: -212.1769 

 G-structure:  ~animal

       post.mean  l-95% CI u-95% CI eff.samp
animal   0.01376 0.0008889  0.03884    20000

               ~Index

      post.mean  l-95% CI u-95% CI eff.samp
Index  0.002158 0.0005867 0.004329    20000

               ~Study_ID

         post.mean l-95% CI u-95% CI eff.samp
Study_ID  0.006496 0.001691  0.01258    20000

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units  0.002685  0.00165 0.003889    20000

 Location effects: genCV ~ factor(Sex) * factor(GenVar_estimate) 

                                                          post.mean l-95% CI
(Intercept)                                                 0.13459 -0.03708
factor(Sex)genCV_male                                       0.00703 -0.03142
factor(GenVar_estimate)narrow-sense                         0.00562 -0.11312
factor(Sex)genCV_male:factor(GenVar_estimate)narrow-sense   0.01545 -0.03128
                                                          u-95% CI eff.samp
(Intercept)                                                0.30232    19947
factor(Sex)genCV_male                                      0.04259    20963
factor(GenVar_estimate)narrow-sense                        0.12842    20000
factor(Sex)genCV_male:factor(GenVar_estimate)narrow-sense  0.06351    21073
                                                          pMCMC
(Intercept)                                               0.109
factor(Sex)genCV_male                                     0.700
factor(GenVar_estimate)narrow-sense                       0.930
factor(Sex)genCV_male:factor(GenVar_estimate)narrow-sense 0.512</code></pre>
</div>
</div>
<div id="references" class="section level3">
<h3>References</h3>
<p>Berger, D., Grieshop, K., Lind, M.I., Goenaga, J., Maklakov, A.A. &amp; Arnqvist, G. (2014). Intralocus sexual conflict and environmental stress. Evolution, 68, 2184-2196.</p>
<p>de Villemereuil, P. &amp; Nakagawa, S. (2014). General quantitative genetic methods for comparative biology. In: Modern phylogenetic comparative methods and their application in evolutionary biology (ed. Garamszegi, LZ). Springer-Verlag Berlin Heidelberg, p. 552.</p>
<p>Fox, C., Bush, M., Roff, D. &amp; Wallin, W. (2004). Evolutionary genetics of lifespan and mortality rates in two populations of the seed beetle, Callosobruchus maculatus. Heredity, 92, 170-181.</p>
<p>Hadfield, J.D. (2010). MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package. Journal of Statistical Software, 33, 1-22.</p>
<p>Holman, L. &amp; Jacomb, F. (2017). The effects of stress and sex on selection, genetic covariance, and the evolutionary response. J. Evol. Biol., 30, 1898-1909.</p>
<p>Janicke, T., Häderer, I.K., Lajeunesse, M.J. &amp; Anthes, N. (2016). Darwinian sex roles confirmed across the animal kingdom. Science Advances, 2, e1500983.</p>
<p>Nakagawa, S., Poulin, R., Mengersen, K., Reinhold, K., Engqvist, L., Lagisz, M. et al. (2015). Meta-analysis of variation: ecological and evolutionary applications and beyond. Methods in Ecology and Evolution, 6, 143-152.</p>
<p>Nakagawa, S. &amp; Schielzeth, H. (2013). A general and simple method for obtaining R2 from generalized linear mixed-effects models. Methods in Ecology and Evolution, 4, 133-142.</p>
<p>Orme, D., Freckleton, R., Thomas, G., Petzoldt, T., Fritz, S., Isaac, N. et al. (2018). caper - Comparative Analyses of Phylogenetics and Evolution in R. Version 1.0.1.</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<div class="sourceCode" id="cb118"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb118-1" title="1"><span class="kw">sessionInfo</span>()</a></code></pre></div>
<pre><code>R version 4.0.0 (2020-04-24)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=German_Germany.1252  LC_CTYPE=German_Germany.1252   
[3] LC_MONETARY=German_Germany.1252 LC_NUMERIC=C                   
[5] LC_TIME=German_Germany.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] reshape2_1.4.4   ggplot2_3.3.3    lattice_0.20-41  matrixcalc_1.0-3
 [5] tidyr_1.1.3      caper_1.0.1      mvtnorm_1.1-1    MASS_7.3-51.5   
 [9] MCMCglmm_2.32    coda_0.19-4      Matrix_1.2-18    ape_5.4-1       
[13] workflowr_1.6.2 

loaded via a namespace (and not attached):
 [1] tidyselect_1.1.0  xfun_0.22         purrr_0.3.4       colorspace_2.0-0 
 [5] vctrs_0.3.6       generics_0.1.0    htmltools_0.5.1.1 yaml_2.2.1       
 [9] utf8_1.2.1        rlang_0.4.10      later_1.2.0       pillar_1.5.1     
[13] withr_2.4.1       glue_1.4.2        DBI_1.1.1         plyr_1.8.6       
[17] lifecycle_1.0.0   stringr_1.4.0     munsell_0.5.0     gtable_0.3.0     
[21] evaluate_0.14     knitr_1.31        httpuv_1.6.1      parallel_4.0.0   
[25] fansi_0.4.2       highr_0.8         Rcpp_1.0.6        corpcor_1.6.9    
[29] scales_1.1.1      promises_1.2.0.1  fs_1.5.0          tensorA_0.36.2   
[33] digest_0.6.27     stringi_1.5.3     dplyr_1.0.5       grid_4.0.0       
[37] rprojroot_2.0.2   tools_4.0.0       magrittr_2.0.1    tibble_3.1.0     
[41] crayon_1.4.1      whisker_0.4       pkgconfig_2.0.3   ellipsis_0.3.1   
[45] assertthat_0.2.1  rmarkdown_2.7     cubature_2.0.4.1  R6_2.5.0         
[49] nlme_3.1-147      git2r_0.28.0      compiler_4.0.0   </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
